---
layout: page
title: Tutorials
header: Tutorials
group: navigation
---

Workshop and tutorial events are listed [here](http://manuals.bioinformatics.ucr.edu/workshops){:target="_blank"}.

<br/>

## Archive of Past Events

<br/>

<table border="1" valign="top">
<tbody>
<tr style="background-color:rgb(11,83,148)">
    </tr>
<tr>
        
<th style="width:50px;height:16px">Date</th>

        
<th style="width:99px;height:16px">Time</th>
        
<th style="width:79px;height:16px">Location</th>
        
<th style="width:116px;height:16px">Instructors</th>
        
<th style="width:105px;height:16px">Institution</th>
        
<th style="width:528px;height:16px">Title and Description</th>
    </tr>
<tr>
<td valign="top"><span>July 22, 2015</span></td>
<td valign="top"><span>1:00pm - 3:45pm</span></td>
<td valign="top"><a href="https://www.fredhutch.org/en/contact-us.html">Fred Hutchinson Cancer Research Center - Seattle, WA</a></td>
<td valign="top">Thomas Girke</td>
<td valign="top">UCR</td>
<td valign="top">
<div><span><b>Automated NGS workflows with systemPipeR </b>&nbsp;(<a href="http://www.bioconductor.org/help/course-materials/2015/BioC2015/">Annual Bioconductor Conference 2015</a>)</span></div>
<div><u>Description:</u>&nbsp;<span>This tutorial introduces systemPipeR, an R package designed for building end-to-end analysis pipelines with automated report generation for next generation sequence (NGS) applications. The package also provides support for running command-line software, such as NGS aligners and variant callers, on both single machines and compute clusters. The first part of the tutorial introduces the basic design of the package, and the second part gives an overview of a typical VAR-Seq analysis workflow including read QC/preprocessing, variant aware read alignments, variant calling, and annotations of SNPs and indels with genomic context information.&nbsp;</span></div>
<div>Teaching Material:&nbsp;<a href="https://htmlpreview.github.io/?https://raw.githubusercontent.com/tgirke/systemPipeR/master/inst/extdata/slides/systemPipeRslides.html">Slides</a>&nbsp; &nbsp;<a href="https://htmlpreview.github.io/?https://github.com/tgirke/systemPipeR/blob/master/vignettes/systemPipeR.html">Manual</a></div>
<div>Maximum number of participants: 35</span></div>
</td>
</tr>
<tr>
<td>Mar 12, 2015</td>
<td>8:30-5:00pm</td>
<td>&nbsp;MCBIOS-2015 Workshop</td>
<td>Thomas Girke</td>
<td>Marriott Conference Center, Little Rock, AR</td>
<td><span><b>NGS Data Analysis with R/Bioconductor</b><br>
<u>Description:</u></span><span>&nbsp;Introduction to R/Bioconductor; Basics on Analyzing Next Generation Sequencing Data (Part I); Basics on Analyzing Next Generation Sequencing Data (Part II), and Analysis of RNA-Seq Data.<br>
<a href="http://manuals.bioinformatics.ucr.edu/workshops/mcbios">Schedule and Teaching Material</a><br>
</span><span>Maximum number of participants: 45</span></td>
</tr>
<tr>
<td><span>Dec 5, 2014</span></td>
<td><span>9:00am - 12:00pm</span></td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td>
<div><span><b>Introduction to R</b></span></div>
<div><u>Description:</u>&nbsp;R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all experimental and quantitative scientific areas. The associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>&nbsp;project provides access to over 900 R extension packages for the analysis of modern biological and biomedical data sets, such as next generation sequences, comparative genomics, network modeling and statistical analysis. The R software is free and runs on all common operating systems. This workshop module provides an elementary-level introduction into the R environment to equip users with the knowledge required for the subsequent events of this R workshop series. The following topics will be covered: (1) command syntax, (2) basic functions, (3) data import/export, (4) data/object types, (5) graphical display, (6) usage of R packages/libraries (e.g. Bioconductor) and (7) using R for basic data analysis operations.</div>
<div><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:13.3333px;line-height:17.0667px">Schedule and Teaching Material</a></div>
<div>Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 5, 2014</span>&nbsp;</td>
<td><span style="font-size:13.3333px;line-height:17.0667px">1:00pm - 6:00pm</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td>
<div><span style="font-size:13.3333px;line-height:17.0667px"><b>Programming in R</b></span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><u>Description:</u>&nbsp;In recent years the&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R language</a>&nbsp;has become the&nbsp;<i><a href="http://bioinfsurvey.org/analysis/programming_languages/">Lingua Franca</a></i>&nbsp;of data intensive research, and is now by far the most widely used data analysis programming language in bioinfomatics. One of the outstanding strengths of the R language is the ease of programming extensions to automate the analysis and mining of almost any data type. This workshop provides an overview of the basic knowledge for writing beginner level functions and programs in R. The following topics will be introduced: (1) conditional executions, (2) loops, (3) writing custom functions, (4) calling external software, (5) running and debugging R programs, and (6) building custom R packages. Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;</div>
<div><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:13.3333px;line-height:17.0667px">Schedule and Teaching Material</a></div>
<div style="font-size:13.3333px;line-height:17.0667px">Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 6, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">9:00am - 12:30pm</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">Genomics Lecture Hall</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">Thomas Girke</span></td>
<td>UCR</td>
<td>
<div><span style="font-size:13.3333px;line-height:17.0667px"><b>Basics on Analyzing Next Generation Sequencing Data with R/Bioconductor</b></span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><u>Description:</u>&nbsp;R and Bioconductor provide extensive utilities for analyzing sequence data from traditional and next generation sequencing technologies (<i>e.g.</i>&nbsp;Sanger or Illumina). This workshop module will cover the following topics: (1) basics on handling sequence, base call quality and annotation ranges in R; (2) demultiplexing of pooled samples;&nbsp;(3) generation of detailed quality reports of FASTQ files; (4) quality/adaptor trimming and filtering of reads;&nbsp;(5)&nbsp;parsing sequences by annotation ranges;&nbsp;(6) handling alignment and coverage objects such as SAM/BAM files in R; (7) interacting with external programs from R, such as short read aligners and peak callers; (8) read count and density analysis; and (9) genome visualization routines.&nbsp;Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;</div>
<div style="font-size:13.3333px;line-height:17.0667px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:13.3333px;line-height:17.0667px">Schedule and Teaching Material</a></div>
<div style="font-size:13.3333px;line-height:17.0667px">Maximum number of participants: 75<span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span></div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 6, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">1:30pm - 6:00pm</span></td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td>
<div><span style="font-size:13.3333px;line-height:17.0667px"><b>Analysis of RNA-Seq Data with R/Bioconductor</b></span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><u>Description:</u>&nbsp;This workshop will cover the most common RNA-Seq data analysis routines. It will include the following topics: (1) read mapping with intron/splice junction aware aligners; (2) generation of read count data for genes, exons or other genome annotation ranges; (3) normalization methods of read count data; (4) statistical tests for identifying differentially expressed genes (DEGs);&nbsp;(5) enrichment analysis of GO terms; and&nbsp;(7) visualization of read pileups using R graphics and the&nbsp;<a href="http://www.broadinstitute.org/igv/">IGV genome browser</a>; and (6) enrichment analysis of GO terms. Basic knowledge of the R software including sequence handling routines, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-">Basic NGS</a>&nbsp;tutorials, will be expected in this workshop.</div>
<div style="font-size:13.3333px;line-height:17.0667px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="line-height:1.6;font-size:10pt;background-color:transparent">Schedule and Teaching Material</a>&nbsp;</div>
<div style="font-size:13.3333px;line-height:17.0667px">Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 7, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">9:00pm - 12:30pm</span></td>
<td><span style="font-size:13.3333px;line-height:21.3333px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td>
<div><span style="font-size:13.3333px;line-height:17.0667px"><b>Analysis of ChIP-Seq Data with R/Bioconductor</b></span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><u>Description:</u>&nbsp;This workshop will cover the most common ChIP-Seq data analysis routines. It will include the following topics: (1) short read alignment against reference genomes; (2) efficient handling of read coverage data; (3) peak calling with various algorithms implemented in R; (4) integration of data from external peak callers; (5) annotating peaks with genomic context information; (5) statistical analysis of differential binding; (6) peak viewing using R graphics,&nbsp;<a href="http://www.bioconductor.org/packages/release/bioc/html/Gviz.html" style="line-height:1.6;font-size:10pt;background-color:transparent">Gviz</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;and&nbsp;the&nbsp;</span><a href="http://www.broadinstitute.org/igv/" style="line-height:1.6;font-size:10pt;background-color:transparent">IGV genome browser</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">; (7) identification of enriched DNA motifs in peak sequences.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="line-height:1.6;font-size:10pt;background-color:transparent">Introduction to R</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">" and&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-" style="line-height:1.6;font-size:10pt;background-color:transparent">Basic NGS</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;tutorials, will be expected in this workshop.&nbsp;</span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:13.3333px;line-height:17.0667px">Schedule and Teaching Material</a><span style="font-size:13.3333px;line-height:17.0667px">&nbsp;</span></div>
<div style="font-size:13.3333px;line-height:17.0667px">Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 7, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">1:30pm - 6:00pm</span></td>
<td><span style="font-size:13.3333px;line-height:21.3333px">Genomics Lecture Hall</span></td>
<td>Neerja Katiyar</td>
<td>UCR</td>
<td>
<div><span style="font-size:13.3333px;line-height:17.0667px"><b>Analysis of SNP/Var-Seq Data with R/Bioconductor</b></span></div>
<div style="font-size:13.3333px;line-height:17.0667px"><u>Description:</u>&nbsp;This workshop will cover the most common SNP/Var-Seq data analysis routines. It will include the following topics: (1) read mapping with variant aware aligners (e.g. BWA, GSNAP); (2) SNP/indel calling (VariantTools, GATK, BCFtools); (3) handling of standard variant data formats such as VCF; (4)&nbsp;annotating variants with genomic context information including&nbsp;variant mapping to genes, intergenic regions; (5) identification of&nbsp;synonymous/non-synonymous SNPs; (6) variant&nbsp;viewing using R graphics (ggbio) and&nbsp;the&nbsp;<a href="http://www.broadinstitute.org/igv/">IGV genome browser</a>.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-">Basic NGS</a>&nbsp;tutorials, will be expected in this workshop.&nbsp;</div>
<div style="font-size:13.3333px;line-height:17.0667px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:13.3333px;line-height:17.0667px">Schedule and Teaching Material</a></div>
<div style="font-size:13.3333px;line-height:17.0667px">Maximum number of participants: 75</div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 8, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">9:00am - 12:00am</span></td>
<td><span style="font-size:13.3333px;line-height:21.3333px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><span style="background-color:transparent"><span style="font-size:13.3333px;line-height:17.0667px"><b>Analysis of Drug-like Small Molecules and High-Throughput Screens with R/Bioconductor</b></span><br>
<span style="text-decoration:underline;font-size:13.3333px;line-height:17.0667px">Description:</span></span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">This workshop introduces various R packages useful for analyzing&nbsp;drug-like compound and screening data sets. This includes the packages</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">ChemmineR</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">,</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/ChemmineOB.html" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">ChemmineOB</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">,&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/home/chemminer#TOC-Maximum-Common-Substructure-MCS-Searching" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">fmcsR</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">,</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/eiR.html" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">eiR</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">and</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/bioassayR.html" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">bioassayR</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">. Efficient R functions will be introduced for handling/analyzing SDF/MOL files, interfacing with PubChem, structural similarity searching, clustering of compound libraries with a wide spectrum of algorithms and utilities for managing complex compound data sets. In addition, visualization functions for compound clusters and chemical structures will be introduced. The last part will cover the analysis of large publicly available high-throughput screening data sets like PubChem BioAssay. Basic knowledge of the R software, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">Introduction to R</a><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">" tutorial, will be expected in this workshop.&nbsp;<br>
</span>
<div style="font-size:13.3333px;line-height:17.0667px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="line-height:1.6;font-size:10pt;background-color:transparent">Schedule and Teaching Material</a></div>
<div style="font-size:13.3333px;line-height:17.0667px"><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">Maximum number of participants: 75</span><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;</span><span style="font-size:13.3333px;line-height:17.0667px;background-color:transparent">&nbsp;&nbsp;</span></div>
</td>
</tr>
<tr>
<td><span style="font-size:13.3333px;line-height:17.0667px">Dec 16, 2014</span></td>
<td><span style="font-size:13.3333px;line-height:17.0667px">1:00pm - 5:00pm</span></td>
<td><span style="font-size:13.3333px;line-height:21.3333px">Genomics Lecture Hall</span></td>
<td>Rakesh Kaundal</td>
<td>UCR&nbsp;</td>
<td><span style="font-size:13.3333px;line-height:17.0667px"><b>Web-based Analysis of Next Generation Sequence Dat</b></span><b style="font-size:13.3333px;line-height:17.0667px">a</b><b style="font-size:13.3333px;line-height:17.0667px"><br>
</b><u style="font-size:13.3333px;line-height:21.3333px">Description:</u><span style="font-size:13.3333px;line-height:21.3333px">&nbsp;</span><span style="font-size:13.3333px;line-height:17.0667px">This workshop will introduce basic NGS data analysis routines using the web-based Galaxy environment. It will include the following topics: (1) assessment of read qualities; (2) read trimming and filtering routines; (3) aligning reads to reference genomes; (4) generation of read counts for RNA-Seq data; (5) peak calling for ChIP-Seq experiments and (6) visualization routines of read pileups along with annotation information using the freely and very easy-to-use IGV genome browser from the Broad Institute. The material will be useful for both complete beginners and intermediate users (e.g. attended previous R workshop on NGS data analysis). No special computer knowledge is required for this workshop.<br>
</span><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-5-8-2014" style="font-size:10pt;line-height:1.6;background-color:transparent">Schedule and Teaching Material</a><span style="font-size:13.3333px;line-height:21.3333px"><br>
</span><span style="font-size:13.3333px;line-height:21.3333px">Maximum number of participants: 75</span></td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 12, 2013</span></td>
<td><span style="line-height:21.3333320617676px">9:00am - 12:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Introduction to R</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all experimental and quantitative scientific areas. The associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>&nbsp;project provides access to over 700 R extension packages for the analysis of modern biological and biomedical data sets, such as next generation sequences, comparative genomics, network modeling and statistical analysis. The R software is free and runs on all common operating systems. This workshop module provides an elementary-level introduction into the R environment to equip users with the knowledge required for the subsequent events of this R workshop series. The following topics will be covered: (1) command syntax, (2) basic functions, (3) data import/export, (4) data/object types, (5) graphical display, (6) usage of R packages/libraries (e.g. Bioconductor) and (7) using R for basic data analysis operations.<span style="font-size:10pt;line-height:21.3333320617676px;background-color:transparent">&nbsp;</span></div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013">Schedule and Teaching Material</a></div>
<div style="line-height:21.3333320617676px"><span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 12, 2013</span></td>
<td><span style="line-height:21.3333320617676px">1:00pm - 6:00pm</span></td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Programming in R</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;In recent years the&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R language</a>&nbsp;has become the&nbsp;<i><a href="http://bioinfsurvey.org/analysis/programming_languages/">Lingua Franca</a></i>&nbsp;of data intensive research, and is now by far the most widely used data analysis programming language in bioinfomatics. One of the outstanding strengths of the R language is the ease of programming extensions to automate the analysis and mining of almost any data type. This workshop provides an overview of the basic knowledge for writing beginner level functions and programs in R. The following topics will be introduced: (1) conditional executions, (2) loops, (3) writing custom functions, (4) calling external software, (5) running and debugging R programs, and (6) building custom R packages. Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;</div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013">Schedule and Teaching Material</a></div>
<div style="line-height:21.3333320617676px"><span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 13, 2013</span></td>
<td><span style="line-height:21.3333320617676px">9:00am - 12:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span><span style="line-height:21.3333320617676px">&nbsp;</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Visualizing and Clustering High-Throughput Data with R/Bioconductor</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;<span style="line-height:1.6;font-size:10pt;background-color:transparent">R is one of the most powerful environments for visualizing and clustering scientific data and creating beautiful publication quality graphics in a programmable and highly reproducible manner. This workshop will give an overview of the following topics: (1) introduction to R's base and grid graphics; (2) usage of high-level graphics libraries including&nbsp;</span><a href="http://lmdvr.r-forge.r-project.org/figures/figures.html" style="line-height:1.6;font-size:10pt;background-color:transparent">lattice</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;and&nbsp;</span><a href="http://had.co.nz/ggplot2/" style="line-height:1.6;font-size:10pt;background-color:transparent">ggplot2</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">; (3) data pre-processing for efficient visualization; (4) writing of functions to generate customized graphics and automating image outputs; (5) visualization of genome and next generation sequencing data with&nbsp;</span><a href="http://www.bioconductor.org/packages/release/bioc/html/ggbio.html">ggbio</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;and&nbsp;</span><a href="http://www.bioconductor.org/packages/release/bioc/html/Gviz.html" style="line-height:1.6;font-size:10pt;background-color:transparent">Gviz</a>; (6)&nbsp;<span style="line-height:1.6;font-size:10pt;background-color:transparent">overview of the most common clustering algorithms used for profiling data, such as hierarchical clustering, fuzzy K-means clustering, principal component analysis, multidimensional scaling, biclustering and quality assessment of clustering results.</span></div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="font-size:10pt;line-height:1.6;background-color:transparent">Schedule and Teaching Material</a><span style="font-size:10pt;line-height:1.6;background-color:transparent">&nbsp;</span></div>
<div style="line-height:21.3333320617676px">Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 13, 2013</span></td>
<td><span style="line-height:21.3333320617676px">1:00pm - 4:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span>&nbsp;</td>
<td><span style="font-size:small;line-height:20.7999992370605px">Jordan Hayes &amp;&nbsp;</span><span style="line-height:21.3333320617676px">Thomas Girke</span></td>
<td>UCR</td>
<td><b>Linux Part I: Linux Essentials</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;The majority of bioinformatics software, especially in the next generation sequence analysis field, is only available for Unix/Linux-based operating systems. Basic knowledge about its usage provides free access to the most powerful and up-to-date applications for high-throughput data analysis. The workshop will teach beginners the basic command-line syntax for running applications on large data sets on LINUX systems. During the workshop users will work on IIGB's Linux cluster by logging in remotely from their laptops. The following topics will be covered: (1) overview of the Linux operating system, (2) file system organization, (3) getting around, (4) basic Shell commands and scripts, (5) available software, (6) running software like Bowtie, BWA, BLAST, HMMER, PHYLIP, EMBOSS, etc.&nbsp;&nbsp;</div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:1.6;font-size:10pt;background-color:transparent">Schedule and Teaching Material</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;</span></div>
<div style="line-height:21.3333320617676px"><span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 13, 2013</span></td>
<td><span style="line-height:21.3333320617676px">4:00pm - 6:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td><span style="font-size:small;line-height:20.7999992370605px">Jordan Hayes &amp;&nbsp;</span>Thomas Girke</td>
<td>UCR</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;This seminar-style presentation will provide an introduction into the usage of the different load balancing and parallel computing tools available on IIGB's Linux cluster. A discussion will follow to determine the need for future hardware and software upgrades. &nbsp;&nbsp;<br>
</div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:1.6;font-size:10pt;background-color:transparent">Schedule and Teaching Material</a><span style="line-height:1.6;font-size:10pt;background-color:transparent">&nbsp;</span>&nbsp;</div>
<div style="line-height:21.3333320617676px"><span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 14, 2013</span></td>
<td><span style="line-height:21.3333320617676px">9:00am - 12:30pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Basics on Analyzing Next Generation Sequencing Data with R/Bioconductor</b><br>
<div style="line-height:21.3333320617676px"><u>Description:</u>&nbsp;R and Bioconductor provide extensive utilities for analyzing sequence data from traditional and next generation sequencing technologies (<i>e.g.</i>&nbsp;Sanger or Illumina). This workshop module will cover the&nbsp; following topics: (1) basics on handling sequence, base call quality and annotation ranges in R; (2) demultiplexing of pooled samples;&nbsp;(3) generation of detailed quality reports of FASTQ files; (4) quality/adaptor trimming and filtering of reads;&nbsp;(5)&nbsp;parsing sequences by annotation ranges;&nbsp;(6) handling alignment and coverage objects such as SAM/BAM files in R; (7) interacting with external programs from R, such as short read aligners and peak callers; (8) read count and density analysis; and (9) genome visualization routines.&nbsp;Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;</div>
<div style="line-height:21.3333320617676px"><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:1.6;font-size:10pt;background-color:transparent">Schedule and Teaching Material</a></div>
<div style="line-height:21.3333320617676px"><span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></div>
</td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 14, 2013</span></td>
<td><span style="line-height:21.3333320617676px">1:30pm - 6:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Analysis of RNA-Seq Data with R/Bioconductor</b><br>
<u style="line-height:21.3333320617676px">Description:</u><span style="line-height:21.3333320617676px">&nbsp;This workshop will cover the most common RNA-Seq data analysis routines. It will include the following topics: (1) read mapping with intron/splice junction aware aligners; (2) generation of read count data for genes, exons or other genome annotation ranges; (3) normalization methods of read count data; (4) statistical tests for identifying differentially expressed genes (DEGs);&nbsp;(5) enrichment analysis of GO terms; and&nbsp;(7) visualization of read pileups using R graphics and the&nbsp;</span><a href="http://www.broadinstitute.org/igv/" style="line-height:21.3333320617676px">IGV genome browser</a><span style="line-height:21.3333320617676px">; and (6) enrichment analysis of GO terms. Basic knowledge of the R software including sequence handling routines, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="line-height:21.3333320617676px">Introduction to R</a><span style="line-height:21.3333320617676px">" and&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-" style="line-height:21.3333320617676px">Basic NGS</a><span style="line-height:21.3333320617676px">&nbsp;tutorials, will be expected in this workshop.<br>
</span><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:21.3333320617676px">Schedule and Teaching Material</a><br style="line-height:21.3333320617676px">
<span style="line-height:21.3333320617676px">Maximum number of participants: 75</span></td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 15, 2013</span></td>
<td><span style="line-height:21.3333320617676px">9:00pm - 12:30pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td><span style="line-height:21.3333320617676px">Thomas Girke</span></td>
<td>UCR</td>
<td><b>Analysis of ChIP-Seq Data with R/Bioconductor</b><br>
<u style="line-height:21.3333320617676px">Description:</u><span style="line-height:21.3333320617676px">&nbsp;This workshop will cover the most common ChIP-Seq data analysis routines. It will include the following topics: (1) short read alignment against reference genomes; (2) efficient handling of read coverage data; (3) peak calling with various algorithms implemented in R; (4) integration of data from external peak callers; (5) annotating peaks with genomic context information; (5) statistical analysis of differential binding; (6) peak viewing using R graphics,&nbsp;</span><a href="http://www.bioconductor.org/packages/release/bioc/html/Gviz.html" style="font-size:10pt;line-height:1.6;background-color:transparent">Gviz</a><span style="font-size:10pt;line-height:1.6;background-color:transparent">&nbsp;and&nbsp;the&nbsp;</span><a href="http://www.broadinstitute.org/igv/" style="font-size:10pt;line-height:1.6;background-color:transparent">IGV genome browser</a><span style="font-size:10pt;line-height:1.6;background-color:transparent">; (7) identification of enriched DNA motifs in peak sequences.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="font-size:10pt;line-height:1.6;background-color:transparent">Introduction to R</a><span style="font-size:10pt;line-height:1.6;background-color:transparent">" and&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-" style="font-size:10pt;line-height:1.6;background-color:transparent">Basic NGS</a><span style="font-size:10pt;line-height:1.6;background-color:transparent">&nbsp;tutorials, will be expected in this workshop.&nbsp;<br>
</span><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:21.3333320617676px">Schedule and Teaching Material</a><br style="line-height:21.3333320617676px">
<span style="line-height:21.3333320617676px">Maximum number of participants: 75&nbsp;</span></td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 15, 2013</span></td>
<td><span style="line-height:21.3333320617676px">1:30pm - 6:00pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Analysis of SNP/Var-Seq Data with R/Bioconductor</b><br>
<u style="line-height:21.3333320617676px">Description:</u><span style="line-height:21.3333320617676px">&nbsp;This workshop will cover the most common SNP/Var-Seq data analysis routines. It will include the following topics: (1) read mapping with variant aware aligners (e.g. BWA, GSNAP); (2) SNP/indel calling (VariantTools, GATK, BCFtools); (3) handling of standard variant data formats such as VCF; (4)&nbsp;annotating variants with genomic context information including&nbsp;variant mapping to genes, intergenic regions; (5) identification of&nbsp;synonymous/non-synonymous SNPs; (6) variant&nbsp;viewing using R graphics (ggbio) and&nbsp;the&nbsp;</span><a href="http://www.broadinstitute.org/igv/" style="line-height:21.3333320617676px">IGV genome browser</a><span style="line-height:21.3333320617676px">.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="line-height:21.3333320617676px">Introduction to R</a><span style="line-height:21.3333320617676px">" and&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-" style="line-height:21.3333320617676px">Basic NGS</a><span style="line-height:21.3333320617676px">&nbsp;tutorials, will be expected in this workshop.&nbsp;</span><br>
<a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:21.3333320617676px">Schedule and Teaching Material</a><br>
<span style="line-height:21.3333320617676px">Maximum number of participants: 75&nbsp;</span></td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 16, 2013</span></td>
<td><span style="line-height:21.3333320617676px">9:00am - 12:00am</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td><span style="line-height:21.3333320617676px">Tyler Backman &amp; Thomas Girke</span></td>
<td>UCR</td>
<td><b>Analysis of Drug-like Small Molecules and High-Throughput Screens with R/Bioconductor</b><br>
<u style="line-height:21.3333320617676px">Description:</u><span style="line-height:21.3333320617676px">&nbsp;This workshop introduces various R packages useful for analyzing&nbsp;drug-like compound and screening data sets. This includes the packages&nbsp;</span><a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html" style="line-height:21.3333320617676px">ChemmineR</a><span style="line-height:21.3333320617676px">,&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/ChemmineOB.html" style="line-height:21.3333320617676px">ChemmineOB</a><span style="line-height:21.3333320617676px">,&nbsp;</span><a href="http://manuals.bioinformatics.ucr.edu/home/chemminer#TOC-Maximum-Common-Substructure-MCS-Searching" style="line-height:21.3333320617676px">fmcsR</a><span style="line-height:21.3333320617676px">,&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/eiR.html" style="line-height:21.3333320617676px">eiR</a><span style="line-height:21.3333320617676px">&nbsp;and&nbsp;</span><a href="http://www.bioconductor.org/packages/devel/bioc/html/bioassayR.html" style="line-height:21.3333320617676px">bioassayR</a><span style="line-height:21.3333320617676px">. Efficient R functions will be introduced for handling/analyzing SDF/MOL files, interfacing with PubChem, structural similarity searching, clustering of compound libraries with a wide spectrum of algorithms and utilities for managing complex compound data sets. In addition, visualization functions for compound clusters and chemical structures will be introduced. The last part will cover the analysis of large publicly available high-throughput screening data sets like PubChem BioAssay. Basic knowledge of the R software, as introduced in the "</span><a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-" style="line-height:21.3333320617676px">Introduction to R</a><span style="line-height:21.3333320617676px">" tutorial, will be expected in this workshop.&nbsp;<br>
</span><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="font-size:10pt;line-height:1.6;background-color:transparent">Schedule and Teaching Material</a><br>
<span style="line-height:21.3333320617676px">Maximum number of participants: 75&nbsp;</span></td>
</tr>
<tr>
<td><span style="line-height:21.3333320617676px">Dec 16, 2013</span></td>
<td><span style="line-height:21.3333320617676px">1:30pm - 5:30pm</span></td>
<td><span style="line-height:21.3333320617676px">Genomics Lecture Hall</span></td>
<td><span style="font-size:small;line-height:25px">Neerja Katiyar &amp;&nbsp;</span><span style="line-height:21.3333320617676px">Thomas Girke</span></td>
<td>UCR</td>
<td><span style="background-color:transparent"><b>Web-based Analysis of Next Generation Sequence Data</b><br>
<font size="2" style="text-decoration:underline"><span style="line-height:21.3333320617676px">Description:</span></font></span><span style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">&nbsp;This workshop will introduce basic NGS data analysis routines using the web-based&nbsp;</span><a href="http://main.g2.bx.psu.edu/root" style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">Galaxy</a><span style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">&nbsp;environment. It will include the following topics:&nbsp;(1) assessment of read qualities; (2) read trimming and filtering routines; (3) aligning reads to reference genomes; (4) generation of read counts for RNA-Seq data; (5) peak calling for ChIP-Seq experiments and (6) visualization routines of read pileups along with annotation information using the freely and very easy-to-use&nbsp;</span><a href="http://www.broadinstitute.org/igv/home" style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">IGV</a><span style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">&nbsp;genome browser from the Broad Institute. The material will be useful for both complete beginners and intermediate users (</span><i style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">e.g.</i><span style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent">&nbsp;attended previous R workshop on NGS data analysis). No special computer knowledge is required for this workshop.<br>
</span><a href="http://manuals.bioinformatics.ucr.edu/workshops/dec-12-16-2013" style="line-height:21.3333320617676px">Teaching Material</a><span style="line-height:21.3333320617676px;font-size:10pt;background-color:transparent"><br>
</span>
<div style="line-height:21.3333320617676px">Maximum number of participants: 75&nbsp;</div>
</td>
</tr>
<tr>
<td>July 18, 2013</td>
<td>2:00pm - 5:00am</td>
<td><a href="https://secure.bioconductor.org/SeattleFeb12/">Fred Hutchinson Cancer Research Center - Seattle, WA</a></td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Cheminformatics of Drug-like Small Molecules</b><br>
<u>Description:</u>&nbsp;This lab session will introduce several Bioconductor packages (ChemmineR, fmcsR and eiR) for analyzing drug-like small molecule and high-throughput screening data in R. These packages contain utilities for efficient processing of large numbers of molecules, physicochemical/structural property predictions, structural similarity searching, classification and clustering of compound screening libraries, and bioactivity data with a wide spectrum of algorithms. In addition, they offer visualization functions for compound clusters and chemical structures.&nbsp;<br>
<a href="http://www.bioconductor.org/help/course-materials/2013/BioC2013/">Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 6, 2012</td>
<td>10:00am - 1:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Introduction to R</b><br>
<u>Description:</u>&nbsp;R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all experimental and quantitative scientific areas. The associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>&nbsp;project provides access to over 600 R extension packages for the analysis of modern biological and biomedical data sets, such as next generation sequences, comparative genomics, network modeling and statistical analysis. The R software is free and runs on all common operating systems. This workshop module provides an elementary-level introduction into the R environment to equip users with the knowledge required for the subsequent events of this R workshop series. The following topics will be covered: (1) command syntax, (2) basic functions, (3) data import/export, (4) data/object types, (5) graphical display, (6) usage of R packages/libraries (e.g. Bioconductor) and (7) using R for basic data analysis operations.<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 6, 2012</td>
<td>2:00pm - 6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Programming in R</b><br>
<u>Description:</u>&nbsp;In recent years the&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R language</a>&nbsp;has become the&nbsp;<i><a href="http://bioinfsurvey.org/analysis/programming_languages/">Lingua Franca</a></i>&nbsp;of data intensive research, and is now by far the most widely used data analysis programming language in bioinfomatics. One of the outstanding strengths of the R language is the ease of programming extensions to automate the analysis and mining of almost any data type. This workshop provides an overview of the basic knowledge for writing beginner level functions and programs in R. The following topics will be introduced: (1) conditional executions, (2) loops, (3) writing custom functions, (4) calling external software, (5) running and debugging R programs, and (6) building custom R packages. Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 7, 2012</td>
<td>9:00am - 12:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Microarray Analysis and Clustering with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;The statistics software&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R</a>&nbsp;and the associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>&nbsp;project have become the "gold standard" for microarray data analysis. The environment integrates the most advanced analysis tools that are currently available for this task. In addition,&nbsp;R contains a comprehensive set of functions and libraries for clustering large multidimensional data sets.&nbsp;This workshop will cover the following topics: (1) microarray data import/export using&nbsp;Affymetrix data as sample sets, (2) background correction and normalization procedures, (3) array quality assessment, (4) identification of differentially expressed genes, (4) enrichment analysis of Gene Ontology (GO) terms, and (5) &nbsp;an overview on the usage of the most common clustering algorithms used for profiling data, such as hierarchical clustering, fuzzy K-means clustering, principal component analysis, multidimensional scaling, biclustering and quality assessment of clustering results.&nbsp;Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 7, 2012</td>
<td>1:00pm - 4:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Grant Brady &amp; Thomas Girke</td>
<td>UCR</td>
<td><b>Linux Part I: Linux Essentials</b><br>
<u>Description:</u>&nbsp;The majority of bioinformatics software, especially in the next generation sequence analysis field, is only available for Unix/Linux-based operating systems. Basic knowledge about its usage provides free access to the most powerful and up-to-date applications for high-throughput data analysis. The workshop will teach beginners the basic command-line syntax for running applications on large data sets on LINUX systems. During the workshop users will work on IIGB's Linux cluster by logging in remotely from their laptops. The following topics will be covered: (1) overview of the Linux operating system, (2) file system organization, (3) getting around, (4) basic Shell commands and scripts, (5) available software, (6) running software like Bowtie, BWA, BLAST, HMMER, PHYLIP, EMBOSS, etc.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 7, 2012</td>
<td>4:00pm - 6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Grant Brady &amp; Thomas Girke</td>
<td>UCR</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><br>
<u>Description:</u>&nbsp;This seminar-style presentation will provide an introduction into the usage of the different load balancing and parallel computing tools available on IIGB's Linux cluster. A discussion will follow to determine the need for future hardware and software upgrades.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 8, 2012&nbsp;</td>
<td>9:00am - 12:30pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Basics on Analyzing Next Generation Sequencing Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;R and Bioconductor provide extensive utilities for analyzing sequence data from traditional and next generation sequencing technologies (<i>e.g.</i>&nbsp;Sanger or Illumina). This workshop module will cover the following topics: (1) basics on handling sequence, base call quality and annotation ranges in R; (2) demultiplexing of pooled samples;&nbsp;(3) generation of detailed quality reports of FASTQ files; (4) quality/adaptor trimming and filtering of reads;&nbsp;(5)&nbsp;parsing sequences by annotation ranges;&nbsp;(6) handling alignment and coverage objects such as SAM/BAM files in R; (7) interacting with external programs from R, such as short read aligners and peak callers; (8) read count and density analysis; and (9) genome visualization routines.&nbsp;Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a><br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 8, 2012</td>
<td>1:30pm - 6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Analysis of RNA-Seq Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;This workshop will cover the most common RNA-Seq data analysis routines. It will include the following topics: (1) read mapping with intron/splice junction aware aligners; (2) generation of read count data for genes, exons or other genome annotation ranges; (3) normalization methods of read count data; (4) statistical tests for identifying differentially expressed genes (DEGs);&nbsp;(5) enrichment analysis of GO terms; and&nbsp;(7) visualization of read pileups using R graphics and the&nbsp;<a href="http://www.broadinstitute.org/igv/">IGV genome browser</a>; and (6) enrichment analysis of GO terms. Basic knowledge of the R software including sequence handling routines, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-">Basic NGS</a>&nbsp;tutorials, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 9, 2012</td>
<td>9:00pm - 12:30pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Analysis of ChIP-Seq Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;This workshop will cover the most common ChIP-Seq data analysis routines. It will include the following topics: (1) short read alignment against reference genomes; (2) efficient handling of read coverage data; (3) peak calling with various algorithms implemented in R; (4) integration of data from external peak callers; (5) annotating peaks with genomic context information; (5) statistical analysis of differential binding; (6) peak viewing using R graphics&nbsp;and&nbsp;the&nbsp;<a href="http://www.broadinstitute.org/igv/">IGV genome browser</a>; (7) identification of enriched DNA motifs in peak sequences.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-">Basic NGS</a>&nbsp;tutorials, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a>&nbsp;<br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 9, 2012</td>
<td>1:30pm - 6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Rebecca Sun</td>
<td>UCR</td>
<td><b>Analysis of SNP/Var-Seq Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;This workshop will cover the most common SNP/Var-Seq data analysis routines. It will include the following topics: (1) read mapping with variant aware aligners; (2) SNP/indel calling; (3) handling of standard variant data formats such as VCF; (4)&nbsp;annotating variants with genomic context information including&nbsp;variant mapping to genes, intergenic regions; (5) identification of&nbsp;synonymous/non-synonymous SNPs; (6) injecting identified variants into reference genome/proteome; and (7) variant&nbsp;viewing using R graphics and&nbsp;the&nbsp;<a href="http://www.broadinstitute.org/igv/">IGV genome browser</a>.&nbsp;Basic knowledge of the R software including sequence handling routines, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Basics-on-Analyzing-Next-Generation-Sequencing-Data-with-R-Bioconductor-December-9-2012-">Basic NGS</a>&nbsp;tutorials, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a>&nbsp;<br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 10, 2012</td>
<td>8:30am - 11:30am</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman &amp; Thomas Girke</td>
<td>UCR</td>
<td><b>Cheminformatics in R for Analyzing Chemical Genomics High-Throughput Screens</b><br>
<u>Description:</u>&nbsp;This workshop introduces various R packages useful for analyzing&nbsp;drug-like compound and screening data sets. This includes the packages&nbsp;<a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a>,&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/home/chemminer#TOC-Maximum-Common-Substructure-MCS-Searching">fmcsR</a>, eiR and&nbsp;<a href="http://manuals.bioinformatics.ucr.edu/home/chemminer#TOC-Biological-Screen-Analysis">bioassayR</a>. Efficient R functions will be introduced for handling/analyzing SDF/MOL files, interfacing with PubChem, structural similarity searching, clustering of compound libraries with a wide spectrum of algorithms and utilities for managing complex compound data sets. In addition, visualization functions for compound clusters and chemical structures will be introduced. The last part will cover the analysis of large publicly available high-throughput screening data sets like PubChem BioAssay. Basic knowledge of the R software, as introduced in the "<a href="http://manuals.bioinformatics.ucr.edu/workshops#TOC-Introduction-to-R-December-6-2012-">Introduction to R</a>" tutorial, will be expected in this workshop.&nbsp;<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a>&nbsp;<br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>Dec 10, 2012</td>
<td>&nbsp;2:00pm - 6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Rebecca Sun &amp; Thomas Girke</td>
<td>&nbsp;UCR</td>
<td><b>Web-based Analysis of Next Generation Sequence Data</b><br>
<u>Description:</u>&nbsp;This workshop will introduce basic NGS data analysis routines using the web-based&nbsp;<a href="http://main.g2.bx.psu.edu/root">Galaxy</a>&nbsp;environment. It will include the following topics:&nbsp;(1) assessment of read qualities; (2) read trimming and filtering routines; (3) aligning reads to reference genomes; (4) generation of read counts for RNA-Seq data; (5) peak calling for ChIP-Seq experiments and (6) visualization routines of read pileups along with annotation information using the freely and very easy-to-use&nbsp;<a href="http://www.broadinstitute.org/igv/home">IGV</a>&nbsp;genome browser from the Broad Institute. The material will be useful for both complete beginners and intermediate users (<i>e.g.</i>&nbsp;attended previous R workshop on NGS data analysis). No special computer knowledge is required for this workshop.<br>
<a href="https://sites.google.com/a/bioinformatics.ucr.edu/bioinformatics-manuals/workshops/dec-6-10-2012">Schedule and Teaching Material</a>&nbsp;<br>
Maximum number of participants: 75</td>
</tr>
<tr>
<td>July 23-25, 2012</td>
<td>8:00am-6:00pm</td>
<td>NM-AIST</td>
<td>Thomas Girke</td>
<td><a href="http://www.nm-aist.ac.tz/"><font color="#000000"><u>The Nelson Mandela African Institute of Science and Technology (NM-AIST)</u></font>, Tanzania, Africa</a></td>
<td><b>Introduction to Bioinformatics</b><br>
<u>Description: </u>This workshop provides an introductory overview of important bioinformatics data analysis concepts&nbsp;related to&nbsp;genome sequencing, database techniques,&nbsp;structural biology, comparative genomics, next generation sequencing, such as RNA-Seq profiling, and small molecule/drug discovery.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/bioinformatics-workshop---nmaist">Schedule, Slides and Exercises</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Feb 27, 2012&nbsp;</td>
<td>12:00pm-2:30pm</td>
<td><a href="https://secure.bioconductor.org/SeattleFeb12/">FHCRC</a></td>
<td>Thomas Girke</td>
<td><a href="https://secure.bioconductor.org/SeattleFeb12/">Fred Hutchinson Cancer Research Center - Seattle, WA</a></td>
<td><b>ChIP-Seq Analysis with R and Bioconductor</b><br>
<u>Description:</u> This workshop provides an introduction to ChIP-Seq analysis in R including short read alignments, handling of coverage data, peak calling routines, peak annotations, differential peak analysis, peak viewing in a genome browser, and analysis of enriched binding motifs.&nbsp;<br>
<a href="http://www.bioconductor.org/help/course-materials/2012/SeattleFeb2012/">Manual, Slides and Exercises</a>&nbsp;<br>
</td>
</tr>
<tr>
<td>Dec 12, 2011</td>
<td>10:00am-1:00pm</td>
<td>Neill Campbell Science Learning Laboratory, UC Riverside</td>
<td>Tyler Backman &amp; Thomas Girke</td>
<td>UCR</td>
<td><b>Analysis of Small Molecule Data with R and Bioconductor</b><br>
<u>Description:</u>&nbsp;This workshop introduces the&nbsp;<a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a>&nbsp;package for mining drug-like compound and screening data sets. The new version of this R package contains functions for handling/analyzing SDF/MOL files, interfacing with PubChem, structural similarity searching, clustering of compound libraries with a wide spectrum of algorithms and utilities for managing complex compound data sets. In addition, it offers visualization functions for compound clusters and chemical structures. The package is well integrated with the online ChemMine Tools service and allows bidirectional communications&nbsp; between the two services. The integration of chemoinformatic tools with the R programming environment has many advantages, such as easy access to a wide spectrum of statistical methods, machine learning algorithms and graphic utilities. Knowledge of the R software, as introduced in the "Introduction to R" course, will be required for attending this workshop.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/chemminer" rel="nofollow">Manual for this workshop</a><br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 11, 2011</td>
<td>1:30pm-6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke&nbsp;</td>
<td>UCR</td>
<td><b>Analysis of RNA-Seq, ChIP-Seq and SNP-Seq Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;This workshop will apply the knowledge covered in the basic sequence analysis event to the most common applications in the NGS field, including RNA-Seq, ChIP-Seq and SNP-Seq. This includes normalization methods and statistical tests for identifying differentially expressed genes (DEGs), peak calling methods and SNP/Indel calling methods. Basic knowledge of the R software and sequence analysis routines as introduced in the previous tutorials, will be expected in this workshop. Due to the computational demands of analyzing next generation sequencing data with reasonable speed, users will work during this workshop on a Linux cluster.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 11, 2011</td>
<td>9:00pm-12:30pm</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke&nbsp;</td>
<td>UCR</td>
<td><b>Basics on Analyzing Next Generation Sequencing Data with R/Bioconductor</b><br>
<u>Description:</u>&nbsp;R and Bioconductor provide extensive utilities for analyzing sequence data from traditional and next generation sequencing technologies (e.g. Sanger or Illumina). This workshop will cover the following topics: (1) basic sequence and string handling; (2) sequence quality assessment/filtering utilities; (3) adaptor trimming; (4) parsing sequences by location; (5) pairwise and multiple sequence alignments; (6) interacting with external short read alignments programs from R,&nbsp;<i>e.g.</i>&nbsp;BWA, Bowtie; (7) range operations (8) read density analysis; and (9) visualization routines. Basic knowledge of the R software, as introduced in the previous tutorial "Introduction to R", will be expected in this workshop. Due to the computational demands of analyzing next generation sequencing data with reasonable speed, users will work during this workshop on a Linux cluster.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 10, 2011</td>
<td>4:00pm-5:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Chris Webber &amp; Thomas Girke</td>
<td>UCR</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><u><br>
Description:</u>&nbsp;This seminar-style presentation will provide an introduction into the usage of the different load balancing and parallel computing tools available on IIGB's Linux cluster. A discussion will follow to determine the need for future hardware and software upgrades. PI's and users from UCR are invited to attend this event.&nbsp;&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/hpc" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40&nbsp;</td>
</tr>
<tr>
<td>Dec 10, 2011&nbsp;</td>
<td>1:00pm-4:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Linux Part I: Linux Essentials</b><u><br>
Description:</u>&nbsp;The majority of freely available bioinformatics software is designed for Unix/Linux-based operating systems. Basic knowledge about its usage provides free access to the most powerful and&nbsp; up-to-date applications in the field. The workshop will teach beginners the basic command-line syntax for running applications on large data sets on our LINUX servers and clusters from a local Windows, Mac or Linux computer. The following topics will be covered: (1) overview of the Linux operating system, (2) file system organization, (3) getting around, (4) basic Shell commands and scripts, (5) available software, (6) running software like Bowtie, BWA, BLAST, HMMER, PHYLIP, EMBOSS, etc.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/linux-basics" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 10, 2011</td>
<td>9:00am-12:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Clustering and Data Mining in R</b><u><br>
Description:</u>&nbsp;R contains a comprehensive set of functions and libraries for clustering large multidimensional data sets. This course will provide an overview on the usage of the most common clustering techniques in R, such as hierarchical clustering with bootstrap, K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis, multidimensional scaling, biclustering, quality assessment of clustering results, etc. Knowledge of the R software, as introduced in the previous tutorial "Introduction to R", will be required for attending this workshop.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_clustering" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40&nbsp;</td>
</tr>
<tr>
<td>Dec 9, 2011</td>
<td>1:00pm-6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Microarray Analysis with R &amp; Bioconductor</b><u><br>
Description:</u>&nbsp;The statistics software&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R</a>&nbsp;and the associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>&nbsp;project have become the "gold standard" for the analysis of dual color microarrays and Affymetrix chips. The environment integrates the most advanced analysis tools that are currently available for profiling data. All software components are freely available for all operating systems. This workshop will cover the following topics: (1) data import/export, (2) background correction and normalization procedures for Affymetrix and cDNA arrays, (3) array quality inspection, (4) identification of differentially expressed genes, (4) visualization of genomic information, (5) overview on clustering methods, and (6) Gene Ontology (GO) analysis. Basic knowledge of the R software, as introduced in the previous tutorial "Introduction to R", will be expected in this workshop.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#BIOCONPROGRAMS" rel="nofollow">Manual for this workshop</a><br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>Dec 9, 2011</td>
<td>9:00am-12:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Graphics and Data Visualization in R</b><br>
<u>Description:</u>&nbsp;R is one of the most powerful environments for visualizing scientific data and creating beautiful publication quality graphics in a programmable and highly reproducible manner. This workshop will give an overview of the following topics: (1) introduction to R's base and grid graphics; (2) usage of high-level graphics libraries including&nbsp;<a href="http://lmdvr.r-forge.r-project.org/figures/figures.html">lattice</a>&nbsp;and&nbsp;<a href="http://had.co.nz/ggplot2/">ggplot2</a>; (3) data pre-processing for efficient visualization; and (4) writing of functions to generate customized graphics and automating image outputs.&nbsp;&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_graphics" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40&nbsp;</td>
</tr>
<tr>
<td>Dec 8, 2011&nbsp;</td>
<td>2:00pm-6:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Programming in R</b><br>
<u>Description:</u>&nbsp;One of the outstanding strengths of the&nbsp;<a href="http://www.r-project.org/" rel="nofollow">R language</a>&nbsp;is the ease of programming extensions to automate the analysis and mining of almost any type of data. This workshop provides an overview of the basic knowledge for writing beginner level programs in R. The following topics will be introduced: (1) conditional executions, (2) loops, (3) writing functions, (4) techniques for improving speed/memory performance, (5) calling external software, (6) running and debugging R programs, (7) object-oriented programming in R and (8) how to build R packages. Knowledge of the R software, as introduced in the previous tutorial "Introduction to R", will be required for attending this workshop.&nbsp;<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/programming-in-r" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40&nbsp;</td>
</tr>
<tr>
<td>Dec 8, 2011&nbsp;</td>
<td>10:00am-1:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR</td>
<td><b>Introduction to R</b><br>
<u>Description:</u>&nbsp;R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated&nbsp;<a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>&nbsp;project provides access to hundreds of additional R packages for the analysis of modern biological and biomedical data sets, such as microarrays, next generation sequencing data, genome annotations, networks, etc. The R software is completely free and runs on all common operating systems. This workshop provides an elementary-level introduction into the R environment to prepare users with the knowledge required for the subsequent events of this R workshop series. The following topics will be covered by this R introduction: (1) command syntax, (2) basic functions, (3) data import/export, (4) data/object types, (5) graphical display, (6) usage of R packages/libraries (e.g. Bioconductor) and (7) using R for basic data analysis operations.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>&nbsp;<br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>May 29, 2011</td>
<td>2:00pm-6:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Tyler Backman &amp; Thomas Girke </td>
<td>UCR <br>
</td>
<td><b>Analysis of Small Molecule Data with R and Bioconductor</b><br>
<u>Description:</u> 
This workshop introduces the <a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a>
 package for mining drug-like 
compound and screening data sets. The new version of this R package 
contains functions for handling/analyzing SDF/MOL files, interfacing with PubChem,
structural similarity searching, clustering of compound libraries with a
 wide spectrum of algorithms and utilities for managing complex compound
 data sets. In addition, it offers visualization functions for compound 
clusters and chemical structures. The package is well integrated with 
the online ChemMine Tools service and allows bidirectional 
communications 
between the two services. The integration of chemoinformatic tools with 
the R programming environment has many advantages, such as easy access 
to a wide spectrum of statistical methods, machine learning algorithms 
and graphic utilities. Knowledge of the R software, as introduced in the
 "Introduction to R" course, will be required for attending this 
workshop.
        <br>
<a href="http://manuals.bioinformatics.ucr.edu/home/chemminer" rel="nofollow">Manual for this workshop</a><br>
Maximum number of participants: 40</td>
</tr>
<tr>
<td>May 29, 2011<br>
</td>
<td>10:00am-1:00pm <br>
</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke <br>
</td>
<td>UCR <br>
</td>
<td><b>GUI-based Exploration and Visualization of Next Generation Sequence Data</b><br>
<u>Description:</u> This workshop will introduce the basics of aligning 
next generation sequence (NGS) data to reference genomes/transcriptomes 
using cloud/web-based applications from the <a href="http://main.g2.bx.psu.edu/root">Galaxy</a> service. Analysis and visualization of the 
read pileups along with annotation information will be performed in the 
free and very easy-to-use <a href="http://www.broadinstitute.org/igv/home">IGV</a>
 genome browser from the Broad Institute. The material will be useful 
for both complete beginners and intermediate users (<i>e.g.</i> attended 
previous R workshop on NGS data analysis). No special computer knowledge
 is required for this workshop.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/gui-ngs-analysis" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40 <br>
</td>
</tr>
<tr>
<td>May 28, 2011 <br>
</td>
<td>1:00pm-5:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Analysis of RNA-Seq, ChIP-Seq and SNP-Seq Data with R/Bioconductor</b><br>
<u>Description:</u>
This workshop will apply the knowledge covered in the basic sequence 
analysis event to the most common applications in the NGS field, 
including RNA-Seq, ChIP-Seq and SNP-Seq. This includes normalization 
methods and statistical tests for identifying differentially expressed 
genes (DEGs), peak calling methods and SNP/Indel calling methods. Basic 
knowledge of the R
software and sequence analysis routines as introduced in the previous 
tutorials,
will be expected in this workshop. Due to the computational demands of
analyzing next generation sequencing data with reasonable speed, users
will work during this workshop on a Linux cluster. <br>

<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>May 28, 2011<br>
</td>
<td>9:00am-12:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Basic Analysis of Next Generation Sequencing Data with R/Bioconductor</b><br>
<u>Description:</u>
R and Bioconductor provide extensive utilities for analyzing
sequence data from traditional and next generation sequencing 
technologies (e.g. Sanger or Illumina). This workshop will cover the 
following topics:
(1) basic sequence and string handling; (2) sequence quality 
assessment/filtering
utilities; (3) adaptor trimming; (4) parsing sequences by location; (5) 
pairwise and multiple sequence
alignments; (6) interacting with external short read alignments programs
 from R, <i>e.g.</i> BWA, Bowtie; (7) range operations (8) read density analysis; and (9)
visualization routines. Basic knowledge of the R
software, as introduced in the previous tutorial "Introduction to R",
will be expected in this workshop. Due to the computational demands of
analyzing next generation sequencing data with reasonable speed, users
will work during this workshop on a Linux cluster. <br>

<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>

        <br>

Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>May 27, 2011<br>
</td>
<td>4:00pm-5:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Alex Levchuk, Tyler Backman &amp; Thomas Girke</td>
<td>UCR <br>
</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><br>
<u>Description:</u> This seminar-style presentation will provide an introduction into the usage of the
different load balancing and parallel computing tools
available on IIGB's Linux cluster. A discussion will follow to determine the
need for future hardware and software upgrades. PI's and users from UCR are invited to attend this event.&nbsp; <br>


<a href="http://manuals.bioinformatics.ucr.edu/home/hpc" rel="nofollow">Manual for this workshop</a>

        <br>

Maximum number of participants: 40
        <br>
</td>
</tr>
<tr>
<td>May 27, 2011<br>
</td>
<td>1:00pm-4:00pm</td>
<td>&nbsp;</td>
<td>Alex Levchuk, Tyler Backman &amp; Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Linux Part I: Linux Essentials</b><br>
<u>Description:</u> The majority of freely available bioinformatics 
software is designed for Unix/Linux-based operating systems. Basic 
knowledge about its usage provides free access to the most powerful and 
up-to-date applications in the field. The workshop will teach beginners 
the basic command-line syntax for running applications on large data 
sets on our LINUX servers and clusters from a local Windows, Mac or 
Linux computer. The following topics will be covered: (1) overview of 
the Linux operating system, (2) file system organization, (3) getting 
around, (4) basic Shell commands and scripts, (5) available software, 
(6) running software like Bowtie, BWA, BLAST, HMMER, PHYLIP, EMBOSS, 
etc. <br>
<a href="http://manuals.bioinformatics.ucr.edu/home/linux-basics" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>May 27, 2011<br>
</td>
<td>10:00am-12:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Introduction to R</b><br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data analysis operations.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>May 15, 2011 <br>
</td>
<td>2:00pm-5:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Clustering and Data Mining in R</b><br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering with bootstrap, 
K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis, 
multidimensional scaling, biclustering, quality assessment of clustering results, etc. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction to R", 
will be required for attending this workshop.
        <br>

<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_clustering" rel="nofollow">Manual for this workshop</a>
        <br>

Maximum number of participants: 40
        <br>
</td>
</tr>
<tr>
<td>May 15, 2011<br>
</td>
<td>10:00am-1:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Microarray Analysis with R &amp; Bioconductor</b><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
project have become the "gold standard" for the analysis of dual color
microarrays and Affymetrix chips. The environment integrates the most
advanced analysis tools that are currently available for profiling
data. All software components are freely available for all operating
systems. This workshop will cover the following topics: (1) data
import/export, (2) background correction and normalization procedures
for Affymetrix and cDNA arrays, (3) array quality inspection, (4)
identification of differentially expressed genes, (4) visualization of
genomic information, (5) overview on clustering methods, and (6) Gene
Ontology (GO) analysis. Basic knowledge of the R software, as
introduced in the previous tutorial "Introduction to R", will be
expected in this workshop.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#BIOCONPROGRAMS" rel="nofollow">Manual for this workshop</a><br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>May 14, 2011<br>
</td>
<td>2:00pm-5:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Graphics and Data Visualization in R</b><br>
<u>Description:</u> R is one of the most powerful environments for 
visualizing scientific data and creating beautiful publication quality 
graphics in a programmable and highly reproducible manner. This workshop
 will give an overview of the following topics: (1) introduction to R's 
base and grid graphics; (2) usage of high-level graphics libraries 
including <a href="http://lmdvr.r-forge.r-project.org/figures/figures.html">lattice</a> and <a href="http://had.co.nz/ggplot2/">ggplot2</a>;
 (3) data pre-processing for efficient visualization; and (4) writing of
 functions to generate customized graphics and automating image 
outputs.&nbsp; <br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_graphics" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40
        <br>
</td>
</tr>
<tr>
<td>May 14, 2011<br>
</td>
<td>10:00am-1:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Programming in R</b><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
is the ease of programming extensions to automate the analysis and
mining of almost any type of data. This workshop provides an overview
of the basic knowledge for writing beginner level programs in R. The
following topics will be introduced: (1) conditional executions, (2)
loops, (3) writing functions, (4) techniques for improving speed/memory
performance, (5) calling external software, (6) running and debugging R
programs, (7) object-oriented programming in R and (8) how to build R 
packages. Knowledge of the R software, as introduced in the previous
tutorial "Introduction to R", will be required for attending this
workshop. <br>
<a href="http://manuals.bioinformatics.ucr.edu/home/programming-in-r" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40
        <br>
</td>
</tr>
<tr>
<td>May 13, 2011<br>
</td>
<td>2:00pm-6:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Introduction to R</b><br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data analysis operations.<br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Nov 20, 2010<br>
</td>
<td>2:00-5:00pm<br>
</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman &amp; Thomas Girke </td>
<td>UCR<br>
</td>
<td><b>Analysis of Small Molecule Data with R and Bioconductor</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/chemminer" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> 
This workshop introduces the <a href="http://bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a>
 package for mining drug-like 
compound and screening data sets. The new version of this R package 
contains functions for handling/analyzing SDF/MOL files, bioactivity 
data from PubChem,
structural similarity searching, clustering of compound libraries with a
 wide spectrum of algorithms and utilities for managing complex compound
 data sets. In addition, it offers visualization functions for compound 
clusters and chemical structures. The package is well integrated with 
the online ChemMine Tools service and allows bidirectional 
communications 
between the two services. The integration of chemoinformatic tools with 
the R programming environment has many advantages, such as easy access 
to a wide spectrum of statistical methods, machine learning algorithms 
and graphic utilities. Knowledge of the R software, as introduced in the
 "Introduction to R" course, will be required for attending this 
workshop.<br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Nov 20, 2010<br>
</td>
<td>10:00-1:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Introduction to R</b><u><br>
</u><a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data mining operations.<br>
Maximum number of participants: 40
        </td>
</tr>
<tr>
<td>Oct 31, 2010<br>
</td>
<td>10:00-1:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke <br>
</td>
<td>UCR<br>
</td>
<td><b>GUI-based Exploration and Visualization of Next Generation Sequence Data</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/gui-ngs-analysis" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> This workshop will introduce the basics of aligning 
next generation sequence (NGS) data to reference genomes/transcriptomes 
using cloud/web-based applications. Analysis and visualization of the 
read pileups along with annotation information will be performed in the 
free and very easy-to-use <a href="http://www.broadinstitute.org/igv/home">IGV</a>
 genome browser from the Broad Institute. The material will be useful 
for both complete beginners and intermediate users (e.g. attended 
previous R workshop on NGS data analysis). No special computer knowledge
 is required for this workshop. <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Oct 30, 2010<br>
</td>
<td>10:00-4:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Analysis of Next Generation Sequencing Data with R and Bioconductor</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u>
R and Bioconductor provide extensive utilities for analyzing
next generation sequence data (NGS) from technologies such
as Illumina (Solexa). This workshop will cover the following topics:
(1) basic sequence and string handling; (2) quality assessment
utilities; (3) quality filtering; (4) adaptor trimming; (5) sequence
alignments; (6) interacting with external alignments programs from R, <i>e.g.</i>
SOAP, Maq, Bowtie; (7) read density and SNP analysis; and (8)
visualization of genome-scale mapping data. Basic knowledge of the R
software, as introduced in the previous tutorial "Introduction into R",
will be expected in this workshop. Due to the computational demands of
analyzing next generation sequencing data with reasonable speed, users
will work during this workshop on a Linux cluster.<br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Oct 29, 2010<br>
</td>
<td>4:00-5:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Alex Levchuk, Tyler Backman, Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/hpc" rel="nofollow">Manual for this workshop</a>

        <br>
Instructors: Alex Levchuk, Tyler Backman &amp; Thomas Girke (UCR)<br>
<u>Description:</u> This seminar-style presentation will provide an introduction into the usage of the
different load balancing and parallel computing tools
available on IIGB's Linux cluster. A discussion will follow to determine the
need for future hardware and software upgrades. PI's and users from UCR are invited to attend this event.&nbsp; <br>
Maximum number of participants: 40
        </td>
</tr>
<tr>
<td>Oct 29, 2010 <br>
</td>
<td>1:00-4:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Alex Levchuk, Tyler Backman, Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Linux Part I: Linux Essentials</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/linux-basics" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The majority of freely available bioinformatics 
software is designed for Unix/Linux-based operating systems. Basic 
knowledge about its usage provides free access to the most powerful and 
up-to-date applications in the field. The workshop will teach beginners 
the basic command-line syntax for running applications on large data 
sets on our LINUX servers and clusters from a local Windows, Mac or 
Linux computer. The following topics will be covered: (1) overview of 
the Linux operating system, (2) file system organization, (3) getting 
around, (4) basic Shell commands and scripts, (5) available software, 
(6) running software like Bowtie, Soap, BLAST, HMMER, PHYLIP, EMBOSS, 
etc. <br>
Maximum number of participants: 40<br>

        </td>
</tr>
<tr>
<td>Oct 3, 2010<br>
</td>
<td>2:00-5:00pm</td>
<td>Genomics Lecture Hall <br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Clustering and Data Mining in R</b><u><br>
</u><a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_clustering" rel="nofollow">Manual for this workshop</a>
        <br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering with bootstrap, 
K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis, 
multidimensional scaling, biclustering, etc. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop.<br>
Maximum number of participants: 40
<br>
</td>
</tr>
<tr>
<td>Oct 3, 2010<br>
</td>
<td>10:00-1:00pm</td>
<td>Genomics<br>
Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Microarray Analysis with R &amp; Bioconductor</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#BIOCONPROGRAMS" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
project have become the "gold standard" for the analysis of dual color
microarrays and Affymetrix chips. The environment integrates the most
advanced analysis tools that are currently available for profiling
data. All software components are freely available for all operating
systems. This workshop will cover the following topics: (1) data
import/export, (2) background correction and normalization procedures
for Affymetrix and cDNA arrays, (3) array quality inspection, (4)
identification of differentially expressed genes, (4) visualization of
genomic information, (5) overview on clustering methods and (6) Gene
Ontology (GO) analysis. Basic knowledge of the R software, as
introduced in the previous tutorial "Introduction into R", will be
expected in this workshop.<br>
Maximum number of participants: 40
<br>
</td>
</tr>
<tr>
<td>Oct 2, 2010<br>
</td>
<td>10:00-3:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Programming in R</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/programming-in-r" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
is the ease of programming extensions to automate the analysis and
mining of almost any type of data. This workshop provides an overview
of the basic knowledge for writing beginner level programs in R. The
following topics will be introduced: (1) conditional executions, (2)
loops, (3) writing functions, (4) techniques for improving speed/memory
performance, (5) calling external software, (6) running and debugging R
programs, and (7) object-oriented programming in R. Knowledge of the R software, as introduced in the previous
tutorial "Introduction into R", will be required for attending this
workshop. <br>
Maximum number of participants: 40
<br>

        </td>
</tr>
<tr>
<td>Oct 1, 2010<br>
</td>
<td>2:00-6:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR</td>
<td><b>Introduction to R</b><u><br>
</u><a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data mining operations.<br>
Maximum number of participants: 40
        </td>
</tr>
<tr>
<td>Mar 6, 2010<br>
</td>
<td>2:00-5:00pm<br>
</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Searching and Clustering Drug-like Compounds in R</b><br>
<a href="http://bioweb.ucr.edu/ChemMineV2/chemminer/tutorial" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> This workshop introduces the ChemmineR package for 
mining drug-like compound and screening data sets. The R package 
contains functions for structural similarity searching, clustering of 
compound libraries with a wide spectrum of algorithms and utilities for 
managing complex compound data sets. In addition, it offers 
visualization functions for compound clusters and chemical structures. 
The package is well integrated with the online ChemMine database and 
allows bidirectional communications between the two services. The 
integration of chemoinformatic tools with the R programming environment 
has many advantages, such as easy access to a wide spectrum of 
statistical methods, machine learning algorithms and graphic utilities. 
Knowledge of the R software, as introduced in the "Introduction into R" 
course, will be required for attending this workshop.
        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Mar 6, 2010<br>
</td>
<td>10:00-12:00pm</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Clustering and Data Mining in R</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#R_clustering" rel="nofollow">Manual for this workshop</a>
        <br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering with bootstrap, 
K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis, 
multidimensional scaling, biclustering, etc. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop.
        <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Mar 5, 2010</td>
<td>2:00-5:00pm</td>
<td>HMNSS1500 (Humanities)</td>
<td>Thomas Girke<br>
</td>
<td>UCR <br>
</td>
<td><b>Microarray Analysis with R &amp; Bioconductor</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual#BIOCONPROGRAMS" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
project have become the "gold standard" for the analysis of dual color
microarrays and Affymetrix chips. The environment integrates the most
advanced analysis tools that are currently available for profiling
data. All software components are freely available for all operating
systems. This workshop will cover the following topics: (1) data
import/export, (2) background correction and normalization procedures
for Affymetrix and cDNA arrays, (3) array quality inspection, (4)
identification of differentially expressed genes, (4) visualization of
genomic information, (5) overview on clustering methods and (6) Gene
Ontology (GO) analysis. Basic knowledge of the R software, as
introduced in the previous tutorial "Introduction into R", will be
expected in this workshop.<br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Mar 5, 2010</td>
<td>9:00-12:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR</td>
<td><b>Introduction to R</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data mining operations.<br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Feb 26, 2010</td>
<td>2:00-6:00pm<br>
</td>
<td>Genomics Lecture Hall </td>
<td>Tyler Backman, Rebecca Sun &amp; Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Analysis of HT Sequencing Data with R and Bioconductor</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/ht-seq" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u>
R and Bioconductor provide extensive utilities for analyzing
high-throughput sequencing data from next generation technologies, such
as Illumina (Solexa). This workshop will cover the following topics:
(1) basic sequence and string handling; (2) quality assessment
utilities; (3) quality filtering; (4) adaptor trimming; (5) sequence
alignments; (6) interacting with external alignments programs from R, <i>e.g.</i>
SOAP, Maq, Bowtie; (7) read density and SNP analysis; and (8)
visualization of genome-scale mapping data. Basic knowledge of the R
software, as introduced in the previous tutorial "Introduction into R",
will be expected in this workshop. Due to the computational demands of
analyzing next generation sequencing data with reasonable speed, users
will work during this workshop on a Linux cluster. <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Feb 19, 2010<br>
</td>
<td>4:00-5:00pm<br>
</td>
<td>Genomics Lecture Hall </td>
<td>Alex Levchuk, Tyler Backman, Thomas Girke</td>
<td>UCR<br>
</td>
<td><b>Linux Part II: Using IIGB's Linux Cluster</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/hpc" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> This seminar-style presentation will provide an introduction into the usage of the
different load balancing and parallel computing tools
available on IIGB's Linux cluster. A discussion will follow to determine the
need for future hardware and software upgrades. PI's and users from UCR are invited to attend this event.&nbsp;&nbsp; <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Feb 19, 2010<br>
</td>
<td>1:00-4:00pm</td>
<td>Genomics Lecture Hall <br>
</td>
<td>Alex Levchuk, Tyler Backman, Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Linux Part I: Linux Essentials</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/linux-basics" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> The majority of freely available bioinformatics 
software is designed for Unix/Linux-based operating systems. Basic 
knowledge about its usage provides free access to the most powerful and 
up-to-date applications in the field. The workshop will teach beginners 
the basic command-line syntax for running applications on large data 
sets on our LINUX servers and clusters from a local Windows, Mac or 
Linux computer. The following topics will be covered: (1) overview of 
the Linux operating system, (2) file system organization, (3) getting 
around, (4) basic Shell commands and scripts, (5) available software, 
(6) running software like Bowtie, Soap, BLAST, HMMER, PHYLIP, EMBOSS, 
etc. <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Jan 30, 2010</td>
<td>10:00-3:00pm<br>
</td>
<td>Genomics Lecture Hall</td>
<td>Thomas Girke<br>
</td>
<td>UCR<br>
</td>
<td><b>Programming in R</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/programming-in-r" rel="nofollow">Manual for this workshop</a><br>
<u>Description: </u>One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
is the ease of programming extensions to automate the analysis and
mining of almost any type of data. This workshop provides an overview
of the basic knowledge for writing beginner-level programs in R. The
following topics will be introduced: (1) conditional executions, (2)
loops, (3) writing functions, (4) techniques for improving speed/memory
performance, (5) calling external software, (6) running and debugging R
programs. Knowledge of the R software, as introduced in the previous
tutorial "Introduction into R", will be required for attending this
workshop. <br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td>Jan 29, 2010</td>
<td>2:00-6:00pm</td>
<td>Genomics Lecture Hall<br>
</td>
<td>Thomas Girke<br>
</td>
<td>UCR</td>
<td><b>Introduction to R</b><br>
<a href="http://manuals.bioinformatics.ucr.edu/home/R_BioCondManual" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
project provides access to hundreds of additional R packages for the
analysis of modern biological and biomedical data sets, such as
microarrays, next generation sequencing data, genome annotations,
networks, etc. The R software is completely free and runs on all common
operating systems. This workshop provides an elementary-level
introduction into the R environment to prepare users with the knowledge
required for the subsequent events of this R workshop series. The
following topics will be covered by this R introduction: (1) command
syntax, (2) basic functions, (3) data import/export, (4) data/object
types, (5) graphical display, (6) usage of R packages/libraries (e.g.
Bioconductor) and (7) using R for basic data mining operations.<br>
Maximum number of participants: 40<br>
</td>
</tr>
<tr>
<td style="width:50px;height:214px">July 23, 2009</td>
<td style="width:99px;height:214px">2:30-7:00pm <br>
</td>
<td style="width:79px;height:214px">1104 Batchelor Hall </td>
<td style="width:116px;height:214px">Tyler Backman &amp; Thomas Girke</td>
<td style="width:105px;height:214px">UCR<br>
</td>
<td style="width:528px;height:214px"><b>Analysis of High-Throughput Sequencing Data with R and Bioconductor<br>
</b><a href="http://htseq.ucr.edu/HT_BioC_Manual" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> R and Bioconductor provide extensive utilities for
analyzing high-throughput sequencing data from next generation
technologies, such as Illumina (Solexa). This workshop will cover the
following topics: (1) basic sequence and string handling; (2) quality
assessment utilities; (3) quality filtering; (4) adaptor trimming; (5)
sequence alignments; (6) interacting with external alignments programs
from R, <i>e.g.</i> SOAP, Maq, Bowtie; (7) read density and SNP
analysis; and (8) visualization of genome-scale mapping data. Basic
knowledge of the R software, as introduced in the previous tutorial
"Introduction into R", will be expected in this workshop. Due to the
computational demands of analyzing next generation sequencing data with
reasonable speed, users will work during this workshop on a Linux
cluster.<br>
Maximum number of participants: 20<br>
</td>
</tr>
<tr>
<td style="width:50px;height:148px">June 25, 2009</td>
<td style="width:99px;height:148px">&nbsp;2:30-7:00pm</td>
<td style="width:79px;height:148px">1104 Batchelor Hall <br>
</td>
<td style="width:116px;height:148px">Thomas Girke <br>
</td>
<td style="width:105px;height:148px">UCR<br>
</td>
<td style="width:528px;height:148px"><b>Clustering and Data Mining in R<br>
</b><a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html#R_clustering" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R contains a comprehensive set of functions and
libraries for clustering large multidimensional data sets. This course
will provide an overview on the usage of the most common clustering
techniques in R, such as hierarchical clustering with bootstrap,
K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis,
multidimensional scaling, biclustering, etc. Knowledge of the R
software, as introduced in the previous tutorial "Introduction into R",
will be required for attending this workshop.<br>
Maximum number of participants: 20</td>
</tr>
<tr>
<td style="width:50px;height:181px">&nbsp;June 5, 2009</td>
<td style="width:99px;height:181px">&nbsp;2:30-7:00pm</td>
<td style="width:79px;height:181px">1104 Batchelor Hall</td>
<td style="width:116px;height:181px">&nbsp;Thomas Girke<br>
</td>
<td style="width:105px;height:181px">UCR</td>
<td style="width:528px;height:181px"><b>Programming in R<br>
</b><a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a>

        <br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
is the ease of programming extensions to automate the analysis and
mining of almost any type of data. This workshop provides an overview
of the basic knowledge for writing beginner-level programs in R. The
following topics will be introduced: (1) conditional executions, (2)
loops, (3) writing functions, (4) techniques for improving speed/memory
performance, (5) calling external software, (6) running and debugging R
programs. Knowledge of the R software, as introduced in the previous
tutorial "Introduction into R", will be required for attending this
workshop.<br>
Maximum number of participants: 20<br>
</td>
</tr>

    </tbody>
<tbody>
        
<tr>
            
<td>April 24, 2009</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are freely available for all operating systems.
 This workshop will cover the following topics: (1) data import/export, 
(2) background correction and normalization procedures for Affymetrix 
and cDNA arrays, (3) array quality inspection, (4) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) overview on clustering methods and (6) Gene Ontology 
(GO) analysis. Basic knowledge of the R software, as introduced in the 
previous tutorial "Introduction into R", will be expected in this 
workshop.
            <br>
Maximum number of participants: 20</td>
        </tr>

        
<tr>
            
<td>March 26, 2009</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Analysis of High-Throughput Sequencing Data with R and Bioconductor</b><br>
<a href="http://htseq.ucr.edu/" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R and Bioconductor provide extensive utilities for 
analyzing high-throughput sequencing data from next generation 
technologies, such as Illumina (Solexa). This workshop will cover the 
following topics: (1) basic sequence and string handling; (2) quality 
assessment utilities; (3) quality filtering; (4) adaptor trimming; (5) 
sequence alignments; (6) interacting with external alignments programs 
from R, <i>e.g.</i> SOAP, Maq, Bowtie; (7) read density and SNP 
analysis; and (8) visualization of genome-scale mapping data. Basic 
knowledge of the R software, as introduced in the previous tutorial 
"Introduction into R", will be expected in this workshop. Due to the 
computational demands of analyzing next generation sequencing data with 
reasonable speed, users will work during this workshop on a Linux 
cluster.   
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>
            
<td>Feb 24, 2009</td>

            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>) is a versatile data analysis environment that has a broad application spectrum in all science areas. The associated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
 project provides access to hundreds of additional R packages for the 
analysis of modern biological and biomedical data sets, such as 
microarrays, next generation sequencing data, genome annotations, 
networks, etc. The R software is completely free and runs on all common 
operating systems. This workshop provides an elementary-level 
introduction into the R environment to prepare users with the knowledge 
required for the subsequent events of this R workshop series. The 
following topics will be covered by this R introduction: (1) command 
syntax, (2) basic functions, (3) data import/export, (4) data/object 
types, (5) graphical display, (6) usage of R packages/libraries (e.g. 
Bioconductor) and (7) using R for basic data mining operations. 
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>Dec 18, 2008</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Searching and Clustering Drug-like Compounds in R</b><br>
<a href="http://bioweb.ucr.edu/ChemMineV2/chemminer/tutorial" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> This workshop introduces the ChemmineR package for 
mining drug-like compound and screening data sets. The R package 
contains functions for structural similarity searching, clustering of 
compound libraries with a wide spectrum of algorithms and utilities for 
managing complex compound data sets. In addition, it offers 
visualization functions for compound clusters and chemical structures. 
The package is well integrated with the online ChemMine database and 
allows bidirectional communications between the two services. The 
integration of chemoinformatic tools with the R programming environment 
has many advantages, such as easy access to a wide spectrum of 
statistical methods, machine learning algorithms and graphic utilities. 
Knowledge of the R software, as introduced in the "Introduction into R" 
course, will be required for attending this workshop.
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>
            
<td>Nov 25, 2008</td>
            
<td>2:30-7:00pm</td>

            
<td><a href="http://commons.ucr.edu/event-scheduling/spaces.htm#commons" rel="nofollow">Commons Rm 379</a> (<a href="http://www.campusmap.ucr.edu/campusMap.php?loc=COMMS" rel="nofollow">Map</a>)</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Clustering and Data Mining in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html#R_clustering" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering with bootstrap, 
K-means, PAM, fuzzy clustering, QT, SOM, principal component analysis, 
multidimensional scaling, biclustering, etc. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop.
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>Sept 25, 2008</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of data. This workshop provides an overview of
 the basic knowledge for writing beginner-level programs in R. The 
following topics will be introduced: (1) basic syntax, (2) executing R 
programs, (3) calling external software, (4) regular expressions, (5) 
writing functions, (6) control structures (loops) and (7) running R 
programs on Linux clusters. Knowledge of the R software, as introduced 
in the previous tutorial "Introduction into R", will be required for 
attending this workshop.
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>
            
<td>Aug 28, 2008</td>

            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are freely available for all operating systems.
 This workshop will cover the following topics: (1) data import/export, 
(2) background correction and normalization procedures for Affymetrix 
and cDNA arrays, (3) array quality inspection, (4) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) overview on clustering methods and (6) Gene Ontology 
(GO) analysis. Basic knowledge of the R software, as introduced in the 
previous tutorial "Introduction into R", will be expected in this 
workshop.
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>July 30, 2008</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (5) using R for data mining, (6) graphical display and (7) usage 
of R packages and libraries (e.g. BioConductor). 
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>

            
<td>Feb 28, 2008</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Searching and Clustering Drug-like Compounds in R</b><br>
<a href="http://bioweb.ucr.edu/ChemMineV2/chemminer/tutorial" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> This workshop introduces the ChemmineR package for 
mining drug-like compound and screening data sets. The R package 
contains functions for structural similarity searching, clustering of 
compound libraries with a wide spectrum of algorithms and utilities for 
managing complex compound data sets. In addition, it offers 
visualization functions for compound clusters and chemical structures. 
The package is well integrated with the online ChemMine database and 
allows bidirectional communications between the two services. The 
integration of chemoinformatic tools with the R programming environment 
has many advantages, such as easy access to a wide spectrum of 
statistical methods, machine learning algorithms and graphic utilities. 
Knowledge of the R software, as introduced in the "Introduction into R" 
course, will be required for attending this workshop.
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>Jan 31, 2008</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Clustering and Data Mining in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html#CLUSTERBACK" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering, bootstrap, K-means, 
PAM, QT, SOM, principal component analysis, multidimensional scaling, 
etc. Knowledge of the R software, as introduced in the previous tutorial
 "Introduction into R", will be required for attending this workshop.
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>
            
<td>Nov 29, 2007</td>
            
<td>2:30-7:00pm</td>

            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u>  One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of numeric data. This workshop provides an 
overview of the basic knowledge for writing beginner-level programs in 
R. The following topics will be introduced: (1) basic syntax, (2) 
executing R programs, (3) calling external software, (4) regular 
expressions, (5) writing functions, (6) control structures (loops) and 
(7) running R programs on our Linux cluster. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop.  
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>Oct 25, 2007</td>
            
<td>2:30-7:00pm</td>
            
<td>1104 Batchelor Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are freely available for all operating systems.
 This workshop will cover the following topics: (1) data import/export, 
(2) background correction and normalization procedures for Affymetrix 
and cDNA arrays, (3) array quality inspection, (4) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) overview on clustering methods and (6) Gene Ontology 
(GO) analysis. Basic knowledge of the R software, as introduced in the 
previous tutorial "Introduction into R", will be expected in this 
workshop.   
            <br>
Maximum number of participants: 20</td>
        </tr>

        
<tr>
            
<td>Sept 27, 2007</td>
            
<td>2:00-6:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (5) using R for data mining, (6) graphical display and (7) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 20</td>
        </tr>
        
<tr>

            
<td>Sept 27, 2007</td>
            
<td>9:00-1:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (5) using R for data mining, (6) graphical display and (7) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 20</td>

        </tr>
        
<tr>
            
<td>Apr 25, 2007</td>
            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Clustering and Data Mining in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html#CLUSTERBACK" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> R contains a comprehensive set of functions and 
libraries for clustering large multidimensional data sets. This course 
will provide an overview on the usage of the most common clustering 
techniques in R, such as hierarchical clustering, bootstrap, K-means, 
PAM, QT, SOM, principal component analysis, multidimensional scaling, 
etc. Knowledge of the R software, as introduced in the previous tutorial
 "Introduction into R", will be required for attending this workshop.
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>Jan 25, 2007</td>
            
<td>2:00-7:00pm</td>

            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of numeric data. This workshop provides an 
overview of the basic knowledge for writing beginner-level programs in 
R. The following topics will be introduced: (1) basic syntax, (2) 
executing R programs, (3) calling external software, (4) regular 
expressions, (5) writing functions, (6) control structures (loops) and 
(7) running R programs on our Linux cluster. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop.
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Nov 16, 2006</td>
            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u>The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are freely available for all operating systems.
 This workshop will cover the following topics: (1) data import/export, 
(2) background correction and normalization procedures for Affymetrix 
and cDNA arrays, (3) array quality inspection, (4) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) overview on clustering methods and (6) Gene Ontology 
(GO) analysis. Basic knowledge of the R software, as introduced in the 
previous tutorial "Introduction into R", will be expected in this 
workshop.
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>

            
<td>Oct 26, 2006</td>
            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u>The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (5) using R for data mining, (6) graphical display and (7) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>July 27, 2006</td>
            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of numeric data. This workshop provides an 
overview of the basic knowledge for writing beginner-level programs in 
R. The following topics will be introduced: (1) basic syntax, (2) 
executing R programs, (3) calling external software, (4) regular 
expressions, (5) writing functions, (6) control structures (loops) and 
(7) running R programs on our Linux cluster. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop. 
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>June 8, 2006</td>

            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are freely available for all operating systems.
 This workshop will cover the following topics: (1) data import/export, 
(2) background correction and normalization procedures for Affymetrix 
and cDNA arrays, (3) array quality inspection, (4) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) overview on clustering methods and (6) Gene Ontology 
(GO) analysis. Basic knowledge of the R software, as introduced in the 
previous tutorial "Introduction into R", will be expected in this 
workshop.   
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>May 25, 2006</td>
            
<td>2:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (6) using R for data mining, (7) graphical display and (8) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>

            
<td>April 18, 2006</td>
            
<td>8:00-2:00pm</td>
            
<td>Loma Linda University</td>
            
<td>Bioinformatic Specialists</td>
            
<td>NCBI</td>
            
<td><b>NCBI Mini-Course</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/NCBI/LomaLindaFlyer.pdf" rel="nofollow">Flyer for this workshop</a><br>
<u>Description:</u> (A) Making Sense of DNA &amp; Protein Sequence: 
Participants will find a gene within a eukaryotic DNA sequence, predict 
the function of the implied protein product, and find a 3D modeling 
template for this protein sequence using NCBI resources. (B) BLAST 
QuickStart: A practical introduction to the BLAST family of 
sequence-similarity search programs. Participants will perform simple 
and specialized searches and learn creative uses of BLAST programs.
            <br>
Organizer: Aileen Gonzales (LLU)<br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Mar 9, 2006</td>
            
<td>3:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of numeric data. This workshop provides an 
overview of the basic knowledge for writing beginner-level programs in 
R. The following topics will be introduced: (1) basic syntax, (2) 
executing R programs, (3) calling external software, (4) regular 
expressions, (5) writing functions, (6) control structures (loops) and 
(7) running R programs on our Linux cluster. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this workshop. 
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>Jan 26, 2006</td>

            
<td>3:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are free and available for all operating 
systems. This workshop will cover the following topics: (1) data 
import/export, (2) background correction and normalization procedures 
for Affymetrix and cDNA arrays, (3) array quality inspection, (4) 
identification of differentially expressed genes, (4) visualization of 
genomic information, (5) overview on clustering methods and (6) Gene 
Ontology (GO) analysis. Basic knowledge of the R software, as introduced
 in the previous tutorial "Introduction into R", will be expected in 
this workshop.   
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Dec 22, 2005</td>
            
<td>3:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (6) using R for data mining, (7) graphical display and (8) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>

            
<td>Nov 22, 2005</td>
            
<td>3:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Programming in R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_Programming.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> One of the outstanding strengths of the <a href="http://www.r-project.org/" rel="nofollow">R language</a>
 is the ease of programming extensions to automate the analysis and 
mining of almost any type of numeric data. This workshop provides an 
overview of the basic knowledge for writing beginner-level programs in 
R. The following topics will be introduced: (1) basic syntax, (2) 
executing R programs, (3) calling external software, (4) regular 
expressions, (5) writing functions, (6) control structures (loops) and 
(7) running R programs on our Linux cluster. Knowledge of the R 
software, as introduced in the previous tutorial "Introduction into R", 
will be required for attending this work shop. 
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Oct 27, 2005</td>
            
<td>3:00-7:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Microarray Analysis with R &amp; BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The statistics software <a href="http://www.r-project.org/" rel="nofollow">R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project have become the "gold standard" for the analysis of dual color 
microarrays and Affymetrix chips. The environment integrates the most 
advanced analysis tools that are currently available for profiling data.
 All software components are free and available for all operating 
systems. This workshop will cover the following topics: (1) data 
import/export, (2) background correction and normalization procedures 
for Affymetrix and cDNA arrays, (3) array quality inspection, (4) 
identification of differentially expressed genes, (4) visualization of 
genomic information, (5) overview on clustering methods and (6) Gene 
Ontology (GO) analysis. Basic knowledge of the R software, as introduced
 in the previous tutorial "Introduction into R", will be expected in 
this workshop.   
            <br>
Maximum number of participants: 12</td>
        </tr>

            
<tr>
<td>Sept 29, 2005</td>
            
<td>3:00-6:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction to R</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The open source software R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 has revolutionized the statistical data analysis for most bioscience 
and chemistry disciplines. The required time to learn the R software is 
well invested, since the R environment covers an unmatched spectrum of 
statistical tools including an efficient programming language for 
automating time-consuming analysis routines. The fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor</a>
 project contains many additional R packages, in particular for the 
analysis of functional genomics and microarray data. Due to their 
popularity, R and BioConductor are continuously updated and extended 
with the latest analysis tools that are available in the different 
research fields. The R environment is completely free and runs on all 
common operating systems. This workshop provides an elementary-level 
introduction into the R environment covering the following topics: (1) 
command syntax, (2) basic functions, (3) data import/export, (4) data 
types, (6) using R for data mining, (7) graphical display and (8) usage 
of R packages and libraries (e.g. BioConductor).
            <br>
Maximum number of participants: 12</td>

        </tr>
<tr>
            
<td>Jul 7, 2005</td>
            
<td>3:00-6:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Introduction into EMBOSS: A Free Open Source Sequence Analysis Package</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/EMBOSS/EMBOSS_MANUAL.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u>  The only free and comprehensive sequence analysis packages is <a href="http://www.hgmp.mrc.ac.uk/Software/EMBOSS/" rel="nofollow">EMBOSS</a>.
 It contains over 150 very useful command-line tools for analyzing DNA 
and protein sequences including pattern searching, phylogenetic 
analysis, data management, feature predictions, proteomics and more. A 
detailed description of all its applications can be found on this <a href="http://www.hgmp.mrc.ac.uk/Software/EMBOSS/Apps/index.html" rel="nofollow">page</a>.
 The workshop will provide an introduction into the functionality and 
usage of the different EMBOSS modules. Knowledge of the basics UNIX 
commands, as introduced in our 'LINUX Essentials' course, is required 
for attending this workshop.
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>Jun 23, 2005</td>

            
<td>3:00-6:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>LINUX Essentials</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/UNIX/linux_manual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The majority of freely available bioinformatics 
software is designed for UNIX/LINUX-based operating systems. Basic 
knowledge about its usage provides free access to the most powerful and 
up-to-date applications in this field. The workshop will teach beginners
 the basic command-line syntax for running applications on large data 
sets on our LINUX servers and clusters from a local PC, Mac or LINUX 
computer. The following topics will be covered: (1) the power of 
UNIX/LINUX, (2) file system organization, (3) getting around, (4) the 
Shell, (5) available software, (6) How to run software like BLAST, 
HMMER, PHYLIP, EMBOSS, etc.
            <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>May 31, 2005</td>
            
<td>3:00-6:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Fast and User Friendly Chip Analysis with R and BioConductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
<u>Description:</u> The <a href="http://www.r-project.org/" rel="nofollow">statistics software R</a> and the associated <a href="http://www.bioconductor.org/" rel="nofollow">BioConductor project</a>
 simplify and standardize the analysis dual color microarrays and 
Affymetrix chips by integrating the differnt analysis levels into one 
environment. The software is free and available for all operating 
systems. This workshop will cover the following topics: (1) a brief 
introduction into R, (2) background correction and normalization 
procedures for Affymetrix and cDNA arrays, (3) identification of 
differentially expressed genes, (4) visualization of genomic 
information, (5) hierarchical clustering and (6) Gene Ontology (GO) 
analysis.
            <br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>

            
<td>Apr 28, 2005</td>
            
<td>1:00-4:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Expression Profiling Analysis with R and Bioconductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
 R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 is a complete statistical software package and programming language for
 data manipulation, calculation and professional graphical display. The 
fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
 project covers many additional R packages for statistical data analysis
 in biosciences, such as tools for the analysis of SNP and 
transcriptional profiling data derived from SAGE, cDNA microarrays, 
Affymetrix chips, etc. This workshop will be divided into two sections: 
the first part will provide a short introduction into R and the second 
part will focus on the usage of Bioconductor packages for the analysis 
of Affymetrix chips and dual-color microarrays (RMA, GCRMA, LIMMA, SAM, 
etc). <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Mar 29-30, 2005</td>
            
<td>1:00-4:00pm</td>
            
<td>Loma Linda University</td>
            
<td>Bioinformatic Specialists</td>
            
<td>NCBI</td>

            
<td><b>NCBI Field Guide Workshop</b><br>
Schedule: <a href="http://faculty.ucr.edu/%7Etgirke/Documents/NCBI/LomaLindaFlyer.pdf" rel="nofollow">Download Flyer</a><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/NCBI/LomaLindaMap.jpg" rel="nofollow">Directions to lecture and lab rooms</a><br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>Mar 17, 2005</td>
            
<td>2:00-5:30pm</td>

            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Expression Profiling Analysis with R and Bioconductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 is a complete statistical software package and programming language for
 data manipulation, calculation and professional graphical display. The 
fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
 project covers many additional R packages for statistical data analysis
 in biosciences, such as tools for the analysis of SNP and 
transcriptional profiling data derived from SAGE, cDNA microarrays, 
Affymetrix chips, etc. This workshop will be divided into two sections: 
the first part will provide a short introduction into R and the second 
part will focus on the usage of Bioconductor packages for the analysis 
of Affymetrix chips and dual-color microarrays (RMA, GCRMA, LIMMA, SAM, 
etc). <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Feb 22-23, 2005</td>
            
<td>9:00-5:00pm</td>
            
<td>City of Hope</td>
            
<td>Bioinformatic Specialists</td>
            
<td>NCBI</td>

            
<td><b>NCBI Workshops</b><br>
First day: <a href="http://www.infosci.coh.org/corelab/NCBI_Feb22-23_overview.asp?parent=training&amp;child=sequence" rel="nofollow">"A Field Guide to GenBank &amp; NCBI Molecular Biology Resources"</a><br>
Second day: <a href="http://www.infosci.coh.org/corelab/NCBI_Feb22-23_overview.asp?parent=training&amp;child=sequence" rel="nofollow">"Exploring 3D Molecular Structures Using NCBI Tools &amp; NCBI QuickScripts"</a><br>
Sign up fee per computer lab session: $25<br>
Maximum number of UCR participants: 15</td>
        </tr>
        
        
<tr>
            
<td>Oct 29, 2004</td>

            
<td>2:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction into R and Bioconductor</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/R_BioCond/R_BioCondManual.html" rel="nofollow">Manual for this workshop</a><br>
R (<a href="http://www.r-project.org/" rel="nofollow">http://www.r-project.org</a>)
 is a complete statistical software package and programming language for
 data manipulation, calculation and professional graphical display. The 
fully integrated <a href="http://www.bioconductor.org/" rel="nofollow">Bioconductor</a>
 project covers many additional R packages for statistical data analysis
 in biosciences, such as tools for the analysis of SNP and 
transcriptional profiling data derived from SAGE, cDNA microarrays, 
Affymetrix chips, etc. This workshop will be divided into two sections: 
the first part will provide an introduction into the basic R commands 
under Linux and the second part will focus on the usage of Bioconductor 
packages for Affymetrix chip analysis (RMA, GCRMA, QC display, SAM, 
etc). Knowledge of the basics UNIX commands, as introduced in our '<a href="http://faculty.ucr.edu/%7Etgirke/Documents/UNIX/linux_manual.html" rel="nofollow">LINUX Essentials</a>' course, is required for attending this workshop.<br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>July 22, 2004</td>
            
<td>2:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke &amp; Josh Lauricha</td>

            
<td>UCR</td>
            
<td><b>Large-Scale Computing on our Bioinfo LINUX Cluster</b><br>
<a href="http://cache.ucr.edu/%7Etgirke/Documents/Cluster/ClusterUsage.html" rel="nofollow">Manual for this workshop</a><br>
Our facility is currently maintaining a 64-CPU LINUX cluster to 
significantly reduce the running time of computationally expensive 
bioinformatics applications. For instance HMM searches of 100,000 
protein sequences against the Pfam database can be finished on this 
cluster within 3-4 days as opposed to 'impractical' ~200 days on a 
single processor machine! This seminar will provide an introduction into
 the usage of the different load ballancing and parallel computing 
systems that are available on our cluster. A discussion will follow to 
determine the need for future hardware growth and software requirements 
in this area.<br>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>July 8, 2004</td>
            
<td>2:00-5:30pm</td>

            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Introduction into EMBOSS: A Free Open Source Sequence Analysis Package</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/EMBOSS/EMBOSS_MANUAL.html" rel="nofollow">Manual for this workshop</a><br>
The only free and comprehensive sequence analysis packages is <a href="http://www.hgmp.mrc.ac.uk/Software/EMBOSS/" rel="nofollow">EMBOSS</a>.
 It contains over 150 very useful command-line tools for analyzing DNA 
and protein sequences including pattern searching, phylogenetic 
analysis, data management, feature predictions, proteomics and more. A 
detailed description of all its applications can be found on this <a href="http://www.hgmp.mrc.ac.uk/Software/EMBOSS/Apps/index.html" rel="nofollow">page</a>.
 The workshop will provide an introduction into the functionality and 
usage of the different EMBOSS modules. Knowledge of the basics UNIX 
commands, as introduced in our 'LINUX Essentials' course, is required 
for attending this workshop.<br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>June 24, 2004</td>
            
<td>2:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>LINUX Essentials</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/UNIX/linux_manual.html" rel="nofollow">Manual for this workshop</a><br>
The majority of freely available bioinformatics software is designed for
 UNIX/LINUX-based operating systems. Basic knowledge about its usage 
provides free access to the most powerful and up-to-date applications in
 this field. The workshop will teach beginners the basic command-line 
syntax for running applications on large data sets on our LINUX servers 
and clusters from a local PC, Mac or LINUX computer. The following 
topics will be covered:
<ul><li>The power of UNIX/LINUX</li>
<li>File system organization</li>
<li>Getting around</li>
<li>The Shell</li>
<li>Available software</li>
<li>How to run software like <a href="http://www.ncbi.nlm.nih.gov/BLAST/blast_overview.shtml#blastq" rel="nofollow">BLAST</a>, <a href="http://hmmer.wustl.edu/" rel="nofollow">HMMER</a>, <a href="http://evolution.genetics.washington.edu/phylip.html" rel="nofollow">PHYLIP</a>, <a href="http://www.hgmp.mrc.ac.uk/Software/EMBOSS/" rel="nofollow">EMBOSS</a>, etc.</li></ul>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>April 15, 2004</td>
            
<td>2:00-3:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Jennifer Le Page</td>
            
<td>ChemBridge</td>

            
<td><b>ChemBridge &amp; CRL: Integrative Chemistry Solutions for Drug Discovery</b><br>
Topics of presentation:
<ul><li>ChemBridge and CRL Corporate Overview</li>
<li>Discovery Chemistry Products</li>
<li>GPCR and Kinase Targeted Libraries</li>
<li>Discovery Chemistry Services</li>
<li>Models for Strategic Collaborations</li></ul>
Maximum number of participants: 12</td>
        </tr>
        
<tr>
            
<td>Mar 25, 2004<br>
Apr 5, 2004</td>

            
<td>2:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Large Scale Data Management for Biologists with the Database Software <i>MS Access</i></b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/Access/Access.html" rel="nofollow">Manual and execises for this workshop</a><br>
 In response to the high demand, the bioinformatics facility is offering
 a workshop for biologists with interest in learning a user-friendly 
database software for managing complex data sets from DNA array, 
proteomic, large-scale sequencing and other high-throughput 
technologies. MS Access provides a simple but efficient database 
environment for organizing large data sets without knowing any 
programming languages. It is also a very useful tool to pre-design data 
structures for future import into more powerful database engines like 
MySQL, PostgreSQL or Oracle. The software is usually preinstalled on 
every Windows computer with MS Office. A Mac version is not available 
yet. This introductory workshop will cover the following topics: data 
im/export, interoperability with spreadsheet programs like Excel, table 
relationships, filters, queries, calculations, table joining, duplicate 
removal, reports and forms. Active Server Pages (ASP) for designing web 
interfaces will not be covered. <br>
Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Mar 4, 2004</td>
            
<td>2:00-4:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>Chemical Genetics Seminar</b><br>
Discussion of future screens and introduction into our compound database.</td>
        </tr>
        
<tr>
            
<td>Jan 7, 2004</td>
            
<td>1:00-3:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>

            
<td>Azhar Alavi</td>
            
<td>Silicon Genetics</td>
            
<td><b>Introduction into GeneSpring</b><br>
This seminar provides a general introduction into GeneSpring, a data 
mining package for expression profiling. New features of the latest 
version 6.1 will be introduced which include new filtering tools and 
clustering techniques.</td>
        </tr>
        
<tr>
            
<td>Oct 30, 2003<br>
Nov 20, 2003</td>

            
<td>2:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>GCG Basics</b> (<a href="http://faculty.ucr.edu/%7Etgirke/GCG.htm#SeqWeb" rel="nofollow">SeqWeb</a>, <a href="http://faculty.ucr.edu/%7Etgirke/GCG.htm#SeqLab" rel="nofollow">SeqLab</a> &amp; <a href="http://faculty.ucr.edu/%7Etgirke/GCG.htm#Command-Line" rel="nofollow">Command Line</a>)<br>
<a href="http://faculty.ucr.edu/%7Etgirke/GCG_Manual/GCG_Manual.htm" rel="nofollow">Manual &amp; Exercises for this workshop</a><br>
These two workshops cover the basics on how to access GCG from SeqWeb, 
SeqLab and the command line. Since SeqLab is the most powerful and 
user-friendly GCG environment, the workshop focuses on this interface 
and gives an overview on the various sequence analysis tools which are 
available in GCG. This includes pattern searches, multiple alignments, 
phylogenetic trees, remote homology detection with HMMER, 
high-throughput sequence analysis such as BLAST searches in batch mode 
and how to make personal BLASTable sequence databases. Participants are 
encouraged to provide their own real-life problems during the exercise 
section. Maximum number of participants: 12</td>

        </tr>
        
<tr>
            
<td>Oct 21-22, 2003</td>
            
<td>9:00-4:00pm</td>
            
<td>City of Hope</td>
            
<td>Bioinformaticians from NCBI</td>
            
<td>NCBI</td>

            
<td><b>NCBI Workshop</b><br>
<a href="http://www.infosci.coh.org/corelab/training_seq_analysis.htm" rel="nofollow">Schedule</a></td>
        </tr>
        
<tr>
            
<td>Oct 2-3, 2003</td>
            
<td>9:00-4:00pm</td>
            
<td>City of Hope</td>

            
<td>Bioinformaticians from NCBI</td>
            
<td>NCBI</td>
            
<td><b>NCBI Minicourses</b><br>
<a href="http://www.infosci.coh.org/corelab/training_seq_analysis.htm" rel="nofollow">Schedule</a></td>
        </tr>
        
<tr>
            
<td>Wed, July 16, 2003</td>

            
<td>2:00-5:00pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>
            
<td><b>Phred/Phrap/Consed</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/Consed/Consed" rel="nofollow">Manual for this workshop</a></td>
        </tr>

        
<tr>
            
<td>April 17 &amp; May 1, 2003</td>
            
<td>3:00-5:30pm</td>
            
<td>1007 Noel T. Keen Hall</td>
            
<td>Thomas Girke</td>
            
<td>UCR</td>

            
<td><b>UNIX/LINUX Essentials for Beginners</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/Documents/UNIX/linux_manual.html" rel="nofollow">Manual for this workshop</a><br>
Topics: The Power of Unix, File System Organization, Getting Around, The Shell, Running Applications, Text Editors</td>
        </tr>
        
<tr>
            
<td><a name="Hughey"></a>March 7, 2003</td>
            
<td>Seminar: 10:30-12:00am <br>
Workshop: 1:00-4:00pm</td>
            
<td>Seminar: Science Library, Rm 240 <br>
 Workshop: Sproul Hall, Rm 2225</td>

            
<td>Richard Hughey</td>
            
<td>UC Santa Cruz</td>
            
<td><b>An Introduction to Hidden Markov Models (seminar)</b> <br>
 Since their introduction to the biological sequence analysis community,
 profile hidden Markov models have become a standard for 
high-performance sequence search, classification, and alignment.  These 
basic functions can also form core components of protein structure 
prediction and genome analysis.  This tutorial presents an introduction 
to the process of creating and using profile hidden Markov models, 
followed by a discussion of the iterative search methods that enable 
particularly distant remote homology detection.  The emphasis will be on
 gaining a qualitative understanding the underlying technology, with 
only a small amount of mathematics. <br>
<br>
<b>The SAM HMM Software System (hands-on workshop)</b><br>
 The Sequence Alignment and Modeling Software System (SAM) is the HMM 
system developed at UCSC in Haussler and Krogh's seminal work.  It has 
been continuously improved since the introduction of profile HMMs, and 
now forms the core of our protein structure prediction efforts, and is 
used by many other research sites.  In this workshop, we will use the 
SAM programs to create and examine HMMs, align sequences, and search 
databases.  We will also use the SAM web servers for protein structure 
prediction.  Attendees are encouraged to bring their own sequence or 
sequences of interest for building and using HMMs <a href="http://www.cse.ucsc.edu/research/compbio/sam.html" rel="nofollow">(web version).</a></td>

        </tr>
        
<tr>
            
<td><a name="Troutman"></a>Dec 4, 2002</td>
            
<td>9am-4pm</td>
            
<td>Watkins, Rm 2101</td>
            
<td>Mike Troutman</td>
            
<td>Affymerix</td>

            
<td><b>Affymetrix Hands-On Training</b> (limited to 12 participants)<br>
This workshop will cover the Affymetrix image analysis tool MAS 5.0, the data mining software DMT and the database MicroDB.</td>
        </tr>
        
<tr>
            
<td><a name="Jaroszewski"></a>Nov 8-9, 2002</td>
            
<td>9am-4pm</td>
            
<td>UCR Extension</td>

            
<td>Lukasz Jaroszewski &amp; Dimitrios Morikis</td>
            
<td>UCSD &amp; UCR</td>
            
<td><b><a href="http://faculty.ucr.edu/%7Etgirke/Tools.htm#Modeling" rel="nofollow">Protein Modeling Workshop:</a></b><br>
 1. Day: "Protein structure, domain databases, sequence and structure analysis"<br>
 2. Day: "Homology modeling and rational drug design"  </td>
        </tr>

        
<tr>
            
<td>Oct 8-10, 2002</td>
            
<td>8am-2pm</td>
            
<td>City of Hope</td>
            
<td>Bioinformaticians from NCBI</td>
            
<td>NCBI</td>

            
<td><b><a href="http://www.infosci.coh.org/corelab/NCBI_overview.htm" rel="nofollow">NCBI Workshop (Summary):</a></b><br>
 Oct 8th: "A Field Guide to GenBank and NCBI Molecular Biology Resources" <br>
 Oct 9th: "Have a BLAST! A Practical Course on the Basic Local Alignment Search Tool (BLAST) from the NCBI" <br>
 Oct 10th: "Making Sense of DNA and Protein Sequences"</td>
        </tr>
        
<tr>
            
<td>Oct 2, 2002</td>
            
<td>10am-12pm</td>

            
<td>Science Library, Rm 240</td>
            
<td>Kyle O'Connor</td>
            
<td>Affymerix</td>
            
<td><b>A Demonstration of the Affymetrix Data Analysis Software</b></td>
        </tr>
        
<tr>
            
<td>July 24, 2002</td>

            
<td>9am-3pm</td>
            
<td>City of Hope</td>
            
<td>Accelrys</td>
            
<td>Accelrys</td>
            
<td><b>Accelrys Biosequence Analysis Workshop</b> <br>
 DS Gene and SeqWeb (GCG)</td>

        </tr>
        
<tr>
            
<td>July 22, 2002</td>
            
<td>10am-4pm</td>
            
<td>Sproul Hall, Rm 2225</td>
            
<td>InforMax</td>
            
<td>InforMax</td>

            
<td><b>Vector NTI Training</b><br>
 Vector NTI database structure &amp; organization, molecule reports, 
maps, features, primer design, back translation, vector design &amp; 
construction, import, export, BioPlot, AlignX, Contig Express.</td>
        </tr>
        
<tr>
            
<td>May 29, 2002 <br>
 May 22, 2002</td>
            
<td>9am - 12pm</td>

            
<td>Sproul Hall, Rm 2225</td>
            
<td>Thomas Girke</td>
            
<td>Center for Plant Cell Biology, UCR</td>
            
<td><b>GCG Workshop</b><br>
<a href="http://faculty.ucr.edu/%7Etgirke/GCG_Manual/GCG_Manual.htm" rel="nofollow">Manual for this workshop</a><br>
Covers the basics on how to access GCG from SeqWeb, SeqLab and the 
command line. Since SeqLab is the most powerful and user-friendly GCG 
environment, the workshop will focus on this application and give an 
overview on the various sequence analysis tools which are available in 
GCG. This will include high-throughput sequence analysis such as BLAST 
searches in batch mode and how to make personal BLASTable sequence 
databases.</td>
        </tr>
        
<tr>

            
<td>April 9, 2002</td>
            
<td>10am - 12pm</td>
            
<td>Surge Bdg, Rm 284</td>
            
<td>Michael Gribscov</td>
            
<td>San Diego Supercomputer Center</td>
            
<td><b>Genomic analysis of plant protein kinases</b></td>
</tr>
</tbody>
</table>
